<!DOCTYPE html>
<html data-html-server-rendered="true" lang="en" data-vue-tag="%7B%22lang%22:%7B%22ssr%22:%22en%22%7D%7D">
  <head>
    <title>MBM Workshop</title><meta name="gridsome:hash" content="e3dae949e5ece8c6bad45ec3d17557dcf74329ec"><meta data-vue-tag="ssr" charset="utf-8"><meta data-vue-tag="ssr" name="generator" content="Gridsome v0.7.23"><meta data-vue-tag="ssr" data-key="viewport" name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"><meta data-vue-tag="ssr" data-key="format-detection" name="format-detection" content="telephone=no"><meta data-vue-tag="ssr" name="description" content="Participate in our workshop and become part of a new and exciting network of young researchers working in modelling with applications in biology and medicine."><meta data-vue-tag="ssr" property="og:title" content="MBM Workshop"><meta data-vue-tag="ssr" property="og:image" content="https://mbm.systemsbiology.se/logo.png"><meta data-vue-tag="ssr" property="og:description" content="Participate in our workshop and become part of a new and exciting network of young researchers working in modelling with applications in biology and medicine."><meta data-vue-tag="ssr" property="og:url" content="https://mbm.systemsbiology.se"><meta data-vue-tag="ssr" property="twitter:card" content="summary"><meta data-vue-tag="ssr" name="twitter:title" content="MBM Workshop"><meta data-vue-tag="ssr" name="twitter.description" content="Participate in our workshop and become part of a new and exciting network of young researchers working in modelling with applications in biology and medicine."><meta data-vue-tag="ssr" name="twitter:image" content="https://mbm.systemsbiology.se/logo.png"><link data-vue-tag="ssr" rel="icon" href="data:,"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="16x16" href="/assets/static/favicon.ce0531f.ea64ed732704f557e7eebec6565fbcb1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="32x32" href="/assets/static/favicon.ac8d93a.ea64ed732704f557e7eebec6565fbcb1.png"><link data-vue-tag="ssr" rel="icon" type="image/png" sizes="96x96" href="/assets/static/favicon.b9532cc.ea64ed732704f557e7eebec6565fbcb1.png"><link data-vue-tag="ssr" rel="apple-touch-icon" type="image/png" sizes="76x76" href="/assets/static/favicon.f22e9f3.ea64ed732704f557e7eebec6565fbcb1.png"><link data-vue-tag="ssr" rel="apple-touch-icon" type="image/png" sizes="152x152" href="/assets/static/favicon.62d22cb.ea64ed732704f557e7eebec6565fbcb1.png"><link data-vue-tag="ssr" rel="apple-touch-icon" type="image/png" sizes="120x120" href="/assets/static/favicon.1539b60.ea64ed732704f557e7eebec6565fbcb1.png"><link data-vue-tag="ssr" rel="apple-touch-icon" type="image/png" sizes="167x167" href="/assets/static/favicon.dc0cdc5.ea64ed732704f557e7eebec6565fbcb1.png"><link data-vue-tag="ssr" rel="apple-touch-icon" type="image/png" sizes="180x180" href="/assets/static/favicon.7b22250.ea64ed732704f557e7eebec6565fbcb1.png"><link rel="preload" href="/assets/css/3.styles.dd40c7db.css" as="style"><link rel="preload" href="/assets/js/app.cd4409d6.js" as="script"><link rel="preload" href="/assets/js/page--src-pages-index-vue.37ec974d.js" as="script"><link rel="prefetch" href="/assets/js/page--node-modules-gridsome-app-pages-404-vue.760b8b62.js"><link rel="prefetch" href="/assets/js/vendors~page--src-pages-index-vue.95a2f81a.js"><link rel="stylesheet" href="/assets/css/3.styles.dd40c7db.css"><noscript data-vue-tag="ssr"><style>.g-image--loading{display:none;}</style></noscript>
  </head>
  <body >
    <div data-server-rendered="true" id="app"><nav class="bg-mbm-green p-0 fixed z-10 top-0 w-full"><div class="mx-auto container lg:flex lg:justify-start"><div class="lg:mx-0 flex items-center justify-between"><div><a href="/#top" class="inline-flex items-center px-4 h-12 lg:h-16
                  focus:outline-none focus:underline text-gray-100 hover:text-white
                  hover:no-underline hover:bg-mbm-green-dark"><span class="my-auto font-extrabold text-3xl tracking-tight">
            MBM 2022
          </span></a></div><div class="block lg:hidden"><button class="px-4 py-2
                      text-gray-300
                      hover:bg-mbm-green-dark
                      hover:text-white hover:border-white
                      focus:outline-none focus:text-white"><svg viewBox="0 0 24 24" class="h-8 w-8 fill-current "><!----><path fill-rule="evenodd" d="M4 5h16a1 1 0 0 1 0 2H4a1 1 0 1 1 0-2zm0 6h16a1 1 0 0 1 0 2H4a1 1 0 0 1 0-2zm0 6h16a1 1 0 0 1 0 2H4a1 1 0 0 1 0-2z"></path></svg></button></div></div><div class="lg:flex hidden"><a href="/#format" class="block text-lg text-gray-100 px-8 lg:px-4 py-2 lg:inline-flex lg:mt-0
                lg:py-0 lg:items-center lg:h-16
                focus:outline-none focus:underline
                hover:bg-mbm-green-dark hover:text-white">
        Format
      </a><a href="/#program" class="block text-lg text-gray-100 px-8 lg:px-4 py-2 lg:inline-flex lg:mt-0
                lg:py-0 lg:items-center lg:h-16
                focus:outline-none focus:underline
                hover:bg-mbm-green-dark hover:text-white">
        Program
      </a><a href="/#invited-speakers" class="block text-lg text-gray-100 px-8 lg:px-4 py-2 lg:inline-flex lg:mt-0
                lg:py-0 lg:items-center lg:h-16
                focus:outline-none focus:underline
                hover:bg-mbm-green-dark hover:text-white">
        Invited Speakers
      </a><a href="/#talks" class="block text-lg text-gray-100 px-8 lg:px-4 py-2 lg:inline-flex lg:mt-0
                lg:py-0 lg:items-center lg:h-16
                focus:outline-none focus:underline
                hover:bg-mbm-green-dark hover:text-white">
        Talks
      </a><a href="/#posters" class="block text-lg text-gray-100 px-8 lg:px-4 py-2 lg:inline-flex lg:mt-0
                lg:py-0 lg:items-center lg:h-16
                focus:outline-none focus:underline
                hover:bg-mbm-green-dark hover:text-white">
        Posters
      </a><a href="/#registration" class="block text-lg text-gray-100 px-8 lg:px-4 py-2 lg:inline-flex lg:mt-0
                lg:py-0 lg:items-center lg:h-16
                focus:outline-none focus:underline
                hover:bg-mbm-green-dark hover:text-white">
        Registration
      </a><a href="/#sponsor" class="block text-lg text-gray-100 px-8 lg:px-4 py-2 lg:inline-flex lg:mt-0
                lg:py-0 lg:items-center lg:h-16
                focus:outline-none focus:underline
                hover:bg-mbm-green-dark hover:text-white">
        Sponsor
      </a><a href="/#contact" class="block text-lg text-gray-100 px-8 lg:px-4 py-2 lg:inline-flex lg:mt-0
                lg:py-0 lg:items-center lg:h-16
                focus:outline-none focus:underline
                hover:bg-mbm-green-dark hover:text-white">
        Contact
      </a></div></div></nav><div class="container mx-auto mt-16 lg:mt-20 flex flex-wrap items-center px-4"><div id="top" class="h-0" style="bottom:100px;position:relative;"></div><header class="border-b pb-4 border-gray-400"><h1 class="mx-auto lg:mx-0 text-center lg:text-left font-bold
                text-2xl xs:text-3xl sm:text-4xl lg:text-5xl
                sm:w-3/4 md:w-9/12 lg:w-3/4 xl:w-7/12 
                leading-tight sm:tracking-wide">
        Workshop on Modelling in Biology and Medicine
      </h1><div class="lg:flex lg:space-x-8 lg:mt-4"><p class="xs:text-xl sm:text-2xl lg:text-3xl
                  text-center lg:text-left w-full lg:w-1/2 
                  mt-2 xs:mt-3 lg:mt-0">
          Bringing together young researchers in Sweden working on the border
          of mathematics, biology and medicine.
        </p><div class="w-full lg:w-1/2 mt-2 xs:mt-3 lg:mt-0"><p class="xs:text-xl sm:text-2xl lg:text-3xl
                    text-center lg:text-left
                    text-mbm-red-light font-semibold">
            Physical Workshop
          </p><p class="xs:text-xl sm:text-2xl lg:text-3xl
                    text-center lg:text-left"><span class="font-semibold">Linköping 2022 </span></p><p class="xs:text-xl sm:text-2xl lg:text-3xl
                    text-center lg:text-left"><span class="font-semibold">May 23<sup>rd</sup></span> and 
            <span class="font-semibold">24<sup>th</sup></span></p></div></div></header><div class="mt-4 lg:grid lg:grid-flow-row lg:grid-rows-1 lg:grid-cols-2
                lg:border-b lg:pb-4 border-gray-400"><section class="body-text border-b lg:border-b-0 lg:border-r pb-4 lg:pb-0 lg:pr-4 border-gray-400"><h2 class="section-header">The Workshop</h2><p>
          We are pleased to invite you to our <span class="font-semibold">third Workshop on Modelling in Biology and Medicine (MBM 2022)</span>.
        </p><p>
          We aim to gather all young researchers in Sweden working on modelling of biological systems. Our ambition is to give all participating PhD students and PostDocs the opportunity to present their work through an oral presentation or a poster. Further, we wish to provide an insight on how modelling in biology and medicine is practiced in academia and industry.
        </p></section><section class="body-text mt-4 lg:mt-0 lg:pl-4 border-b pb-4 lg:border-b-0 lg:pb-0 border-gray-400"><div id="format" class="h-0" style="bottom:100px;position:relative;"></div><h2 class="section-header">Format</h2><p>
          The workshop will be held in both plenary presentation sessions for larger talks as well as in smaller sessions for e.g. poster presentations and discussions.
        </p><p>
          The exact technicalities will be made available for <span class="font-semibold">registered participants</span> closer to the date of the workshop. 
        </p><p>
          We will keep the workshop physically in Linköping, to promote networking. <span class="font-semibold"></span></p></section></div><div class="mt-4 lg:grid lg:grid-flow-row lg:grid-rows-1 lg:grid-cols-1
                lg:border-b lg:pb-4 border-gray-400 w-full"><section class="body-text"><h2 class="section-header">Venue</h2><p>
          The workshop will be arranged at the University Hospital Campus in Linköping. More specificly, the north entrance, marked out in <a href="https://www.google.se/maps/@58.4022431,15.6211556,19z" target="_blank"><u>Google maps</u></a>, one floor below the entrance at <a href="https://old.liu.se/karta?l=sv&amp;px_type=2&amp;px_id=512" target="_blank"><u>Hugo Theorell</u></a>. 
        </p><p>
          If you come to Linköping by public transport we recomend you to trabvel with <a href="https://www.ostgotatrafiken.se/linkopings-resecentrum/us-norra-entren/?walk=true" target="_blank"><u>Östgötatrafiken</u></a>, going with a bus from the central station &quot;Linköpings resecentrum&quot; to &quot;US norra entrén&quot;. 
        </p><p>
          If you are using other means of transportation, it would be easiest to travel to the coordinate location 58.40230939062588, 15.621568767919038, or see these <a href="/directionsMBMLinkoping.pdf" target="_blank"><u>instructions</u></a>.
        </p><p>
          Some close by living options are <a href="https://www.nordicchoicehotels.se/hotell/sverige/linkoping/quality-hotel-ekoxen/" target="_blank"><u>Hotel Ekoxen</u></a> and <a href="https://lvh.se/en/" target="_blank"><u>Linköping's City Hotell</u></a>. 
        </p></section></div><section class="mt-4 border-b border-gray-400 pb-4 w-full"><div id="program" class="h-0" style="bottom: 100px; position: relative;"></div><h2 class="section-header">Program</h2><div class="flex flex-col lg:flex-row pt-2 pb-4"><div class="px-4 w-full lg:w-1/2 flex flex-col"><h3 class="mt-2 sm:text-lg font-semibold">Day 1</h3><table class="mt-4 table-fixed w-full"><thead><tr><th class="w-1/6 text-center font-semibold">Time</th><th class="w-5/6 px-4 text-left font-semibold">Activity</th></tr></thead><tbody><tr class="border-t border-gray-400"><td class="py-2 text-center">12:30</td><td class="py-2 px-4"><span>
                Opening
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">13.00</td><td class="py-2 px-4"><span class="font-semibold">
                Maria Kjellsson
              </span><span class="text-sm lg:text-base text-gray-600 italic">
                (Invited speaker)
              </span></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">14.00</td><td class="py-2 px-4"><span>
                Sebastian Sten
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">14.20</td><td class="py-2 px-4"><span>
                Gustav Lindwall
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">14.40</td><td class="py-2 px-4"><span>
                Talk (TBA)
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">15.00</td><td class="py-2 px-4"><span class="text-gray-600 italic">
                Break
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">15.30</td><td class="py-2 px-4"><span class="font-semibold">
                Jan Hasenauer
              </span><span class="text-sm lg:text-base text-gray-600 italic">
                (Invited speaker)
              </span></td></tr><tr class="border-t border-gray-400 bg-mbm-green-light"><td class="py-2 text-center">16.00</td><td class="py-2 px-4"><span>
                Poster session
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">17.30</td><td class="py-2 px-4"><span>
                End of day
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">18.00</td><td class="py-2 px-4"><span class="text-gray-600 italic">
                Dinner
              </span><!----></td></tr></tbody></table></div><div class="px-4 w-full lg:w-1/2 flex flex-col"><h3 class="mt-2 sm:text-lg font-semibold">Day 2</h3><table class="mt-4 table-fixed w-full"><thead><tr><th class="w-1/6 text-center font-semibold">Time</th><th class="w-5/6 px-4 text-left font-semibold">Activity</th></tr></thead><tbody><tr class="border-t border-gray-400"><td class="py-2 text-center">08.00</td><td class="py-2 px-4"><span>
                Opening
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">08.15</td><td class="py-2 px-4"><span class="font-semibold">
                Tornbjörn Norling
              </span><span class="text-sm lg:text-base text-gray-600 italic">
                (Invited speaker)
              </span></td></tr><tr class="border-t border-gray-400 bg-mbm-green-light"><td class="py-2 text-center">09.00</td><td class="py-2 px-4"><span>
                Poster session
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">10.00</td><td class="py-2 px-4"><span class="text-gray-600 italic">
                Break
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">10.20</td><td class="py-2 px-4"><span class="font-semibold">
                Matthias König
              </span><span class="text-sm lg:text-base text-gray-600 italic">
                (Invited speaker)
              </span></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">11.00</td><td class="py-2 px-4"><span>
                Talk (TBA)
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">11.20</td><td class="py-2 px-4"><span>
                Sonja Mathias
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">11.40</td><td class="py-2 px-4"><span>
                Oscar Arrestam
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">12.00</td><td class="py-2 px-4"><span class="text-gray-600 italic">
                Lunch break
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">13.00</td><td class="py-2 px-4"><span class="font-semibold">
                Andreas Hellander
              </span><span class="text-sm lg:text-base text-gray-600 italic">
                (Invited speaker)
              </span></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">14.00</td><td class="py-2 px-4"><span>
                Talk (TBA)
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">14.20</td><td class="py-2 px-4"><span>
                Lucas Hedström
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">14.40</td><td class="py-2 px-4"><span>
                Talk (TBA)
              </span><!----></td></tr><tr class="border-t border-gray-400"><td class="py-2 text-center">15.00</td><td class="py-2 px-4"><span>
                Closing of Conferance
              </span><!----></td></tr></tbody></table></div></div></section><section class="mt-4 pb-4 border-b border-gray-400"><div id="invited-speakers" class="h-0" style="bottom: 100px; position: relative;"></div><h2 class="section-header">Invited speakers</h2><div class="mt-4 lg:grid lg:grid-flow-row lg:grid-cols-2 lg:gap-x-24"><section class="mt-8"><div class="rounded-lg shadow-md bg-gray-100 px-2 py-2 lg:h-64 lg:pt-4"><div class="ml-2 sm:ml-0 flex items-center space-x-2 sm:justify-center"><div class="rounded-full h-16 w-16 sm:h-20 sm:w-20 lg:h-24 lg:w-24 overflow-hidden"><img alt="Maria Kjellsson" src="data:image/svg+xml,%3csvg fill='none' viewBox='0 0 400 469' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'%3e%3cdefs%3e%3cfilter id='__svg-blur-1ff8e8f3155f1de7c4d3e24ff0bf54e7'%3e%3cfeGaussianBlur in='SourceGraphic' stdDeviation='40'/%3e%3c/filter%3e%3c/defs%3e%3cimage x='0' y='0' filter='url(%23__svg-blur-1ff8e8f3155f1de7c4d3e24ff0bf54e7)' width='400' height='469' xlink:href='data:image/jpeg%3bbase64%2c/9j/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/2wBDAQcHBwoIChMKChMoGhYaKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCj/wAARCABLAEADASIAAhEBAxEB/8QAHAAAAgIDAQEAAAAAAAAAAAAABQYEBwACAwEI/8QALhAAAgEDAwMEAgEDBQAAAAAAAQIDAAQRBRIhMUFhBhMiURRxoVKBkSNC4fDx/8QAGgEAAwEBAQEAAAAAAAAAAAAAAQIDBAAFBv/EAB4RAAICAwEBAQEAAAAAAAAAAAABAhEDITFBEhNR/9oADAMBAAIRAxEAPwBkdyelR5CxOByawtmvYSN/JOQM4HevnarZ763oj7HfdjooySe1CruViuEcKT1x1Apl0izk1NpprhjHa7tqxrwXx3PijMfp%2b0dMCBQB4quPG5iyaiVVNcyxE4Mj481tb6lHKdr/ABPmnrWvSNpPCwjDRSDowNVfc2c9lezRz8%2b2cE/Y%2b6r%2bdE272H3GRkVHkFdrQgwKc5GMZ%2bqyaMqeRQcaEGMOR0qPJfiO6SPgu3xwB91tk4JoPHuGte8ysyxAynaMnjpipVbouv6WfpMASJFAwAOlG4SFwKrjT9VlkuFcLcQvjdsdsgjHimu4vnSxilQ4eUZXNboP48ItfXoUvUD54qp/VsCpqzBhwyHP6pyi1uR2MUlzatKeFQnaSaTfWU7LfQ3JQjbwymkm/QpeAfT2a2m/Hk%2bSngeR2ovPbbYQF5UdPHiorW8Z9sHmPh4pB2B7fqjkY323POMAmp9A9G8ajv0qb6cto21KeVgPkgUf3P8AxQq5coUGcKTzRDQJiLpm7EikgtlRmubSGHaI1AY8dK3eJZpliwpCrtAI6Vw1MXM/stZOBIp715pcd8l2z3QXrnI71rcuIVQVWRtT9H2t0GeUFSRxtwMeR580teuLBIdNC7y7BcZPU1YeoXgEBx1xVaetb3MJ9w4ABPNJlabpHY40rYH9I3IubU2F3yy59pj/ACKYLTdEJIjzkcfsUpaMd0kPtcOMMD5z/wC04MpWUsDyCCP8VF9A1ol3uk3DWchZNvHc0N0e6FtqIjkG5T8f8U/XQXYyeOarzU4fZvcx/wC1iQaEVRykM1vLdR3BMcv%2bmeV3LnH7qV%2bdfGYqBEy/1HIrlo97a3FuokKhgOQfuu1xcwRqTHjPbFXb0V84ZdNsXMj57nnpVUetrg3uqBQcRoPj5/7irAmL3b7UJ29zST6u0tld7oEKijnP12pF0SXCP6cO11HBCdD%2b%2b1GdRu7k3KTRoyIvxjH9X3mlzS3khWPBAx1Pmi9xqjR43ncRyCaR7ewcWiwr2%2bwSq5P3S5fuJS8gHIU8eKm7iXdScgKD/FQZgBBJjvtBro23YHVC1fPJHciaKRk3AZIPfmmbQz%2bXGjSMxPQ5NK8/NrJnnDnFM3pMD2m8NRHTHCC3jhgLbQBj6qp/X2ri7vmsYDiND8sd2%2bv7VdsEaGxyVBOK%2bf8A1MPY1HVXh%2bDC6KgjqATk1phCjPkk2R45LmOMLLEwB6Nj%2ba7fj6hdxFhA6xdnPFc9PvblrOWJpnZEjDqGOcHJ7mphmkmgHuuzYUYyaP5q7IvI%2bH//2Q==' /%3e%3c/svg%3e" width="400" data-src="/assets/static/maria.d042d20.26d4c4d83f4668cc41ef64fd52c4000e.jpg" data-srcset="/assets/static/maria.d042d20.26d4c4d83f4668cc41ef64fd52c4000e.jpg 400w" data-sizes="(max-width: 400px) 100vw, 400px" class="g-image g-image--lazy g-image--loading"><noscript><img src="/assets/static/maria.d042d20.26d4c4d83f4668cc41ef64fd52c4000e.jpg" class="g-image g-image--loaded" width="400" alt="Maria Kjellsson"></noscript></div><h3 class="text-lg sm:text-xl lg:text-2xl font-medium">Maria Kjellsson</h3></div><div class="body-text mt-2 text-gray-700 text-sm md:text-base"><div><p>Maria Kjellsson is an associate professor at the Dept. of Pharmacy at Uppsala University, and leads, together with Mats Karlsson, the research focused on diabetes and associated diseases, performed in the Pharmacometrics group. </p></div></div></div><div class="body-text mt-4"><div><p>The group use mixed-effects models to investigate several complex aspects of diabetes, from pre-clinical experiments to treatment, risk factors, and disease progression of diabetes. The model applications range from drug discovery and drug development to predictions of long-term treatment effects and risk of cardiovascular disease. Currently, Kjellsson is project leader of a Swedish Research Council-sponsored project with the aim to develop a tool to visualize patients' risk of cardiovascular disease and how the risk changes with treatments.</p></div></div></section><section class="mt-8"><div class="rounded-lg shadow-md bg-gray-100 px-2 py-2 lg:h-64 lg:pt-4"><div class="ml-2 sm:ml-0 flex items-center space-x-2 sm:justify-center"><div class="rounded-full h-16 w-16 sm:h-20 sm:w-20 lg:h-24 lg:w-24 overflow-hidden"><img alt="Jan Hasenauer" src="data:image/svg+xml,%3csvg fill='none' viewBox='0 0 128 128' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'%3e%3cdefs%3e%3cfilter id='__svg-blur-53cd048bfd4325de3448929b65535f64'%3e%3cfeGaussianBlur in='SourceGraphic' stdDeviation='40'/%3e%3c/filter%3e%3c/defs%3e%3cimage x='0' y='0' filter='url(%23__svg-blur-53cd048bfd4325de3448929b65535f64)' width='128' height='128' xlink:href='data:image/jpeg%3bbase64%2c/9j/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/2wBDAQcHBwoIChMKChMoGhYaKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCj/wAARCABAAEADASIAAhEBAxEB/8QAGgAAAgMBAQAAAAAAAAAAAAAABgcCAwUEAf/EADMQAAIBAwIEAwcEAQUAAAAAAAECAwAEEQUhBhIxQRNRYQciIzJxgZEzUqHBJUJTcoKx/8QAGgEBAAMBAQEAAAAAAAAAAAAABQIDBAABBv/EACERAAMAAgICAgMAAAAAAAAAAAABAgMRBBIhMQUTQVFh/9oADAMBAAIRAxEAPwBsKanzqilmICqCST2Aqtaz%2bIpWj0iZUODJiPPkCd/4zUafVNkpXZpGTFcSa7cmWTmW3DfBiPZf3H1NEUWjgW4AAAxnpWJoHIu5OAMDNGFvNFJAPAnSQDY8pzihHTtuqPoolY5UyCmuaNGbctyAHHXzrn4C1501STQbyQthC9qx64X5kz6DcemR2oi1aSKSNkWdGcdVBycfSlXfyzWXF9pcwELLBcrJt3Q4DfwTVvHtxfgo5ULJj8j0BqYqpT5dKsU0yBA2h3rI4rt5LrTpIxK0cKxs78vzZGMEHsR/daqVx63G8mmziNOc%2bG%2bQDv8AKf7qnPvo9F/GaWRdvQJaboerS4RtTuI7fwcCeCNGkdv3NzHHpj0rZsbKe0u0itLm9twI3MsxCBpDgAAZBHXLHbyFdnCF4Wtky3ucvertWW/kvvH0y6VWVChV4uZSSfzsKITdLSHPrlMFLOwub2FZRdXNvcq58SRIFkWYY8z033/Iob1y2utLvIZJHhnupiFRXUgKwIOevpjHrTH4TM1hpLW99cCW5LliQnLkn09TQHxrHNf8T2FokcruZCQiDJz12/FTmn2SIXjnq2xzaLdPeaXaXMqhZJYldlHQEjfFaANcGnRC2sbeAbeHGqfgV1hs0zPryAVrb0DifWrV9dx3FUKe1WIeneuPAa0OaK0vrvTnO1vIy4zuVO6n8Guu50VTI8kEU7o/y8s7ADz70O8UoZdXkv8AS5B4qfDYj5XK7Mp%2bm1eWHHsllEIrqymEq7Z2K/WiKxOLeh/j52pVBFpenxaVM9zNFy7YUu5c/wA9KEo9btofabpz3EqRxIH8SRzgLzIQMntXDxBxRfayYodPgdF5sknufL6VZBwwdI0q61fWjmfkZmD9dxjf1OcAdq0cbjVeRP8ARn5fJSht/kdSOGUMpBBGQR0NTDUjPYz7QEt4xw/rsyxxIxFjcSNhQv8AtMfT/SfLbyp2pIGUMpBUjIIOQaQCEJvVPavp0JZNLsp7t%2bzyfDT%2bz/5QhqvGuuasE8e7EMc7%2bFFaW3uAnODzHqR96Gkt0QMT8qjfeo8P28t7rDmBcvEnLGOwkc8q/wAnP2rktvRFsfHAdpZPwTbi9mSBZWeWOSQgYYscY88gDavbXQjeyJAVR/EPwnOPeHoale8FWi6dLJcyyzz2doIbUdEt%2bVQcqO7Erux37dKlqcqXBs9KsvDN3My4blDGEDctv3wDio8rDNZJSQp8bt4Lp16/gXaTwtYaSgcIJLjH6hHT/iO1Lj20auhjXR7U7piWfHr8o/v7imTdXb6Jp0k13dPcWkMZZ3mOZABv17/Q0g7%2b5k1K7ub27/VuJDIw8s9F%2bwwPtWulOKOkhFVWS%2b1PYFSWvhJ4ygYUhs56YPeirSOJ9W0Kb/HXciR53ib3oz/1O1Z17AshaIOBCRhuQbtnsT2%2b35qEkPubVnJI/9k=' /%3e%3c/svg%3e" width="128" data-src="/assets/static/jan.1219de4.550b150914d4a919e4c4613166644168.jpeg" data-srcset="/assets/static/jan.1219de4.550b150914d4a919e4c4613166644168.jpeg 128w" data-sizes="(max-width: 128px) 100vw, 128px" class="g-image g-image--lazy g-image--loading"><noscript><img src="/assets/static/jan.1219de4.550b150914d4a919e4c4613166644168.jpeg" class="g-image g-image--loaded" width="128" alt="Jan Hasenauer"></noscript></div><h3 class="text-lg sm:text-xl lg:text-2xl font-medium">Jan Hasenauer</h3></div><div class="body-text mt-2 text-gray-700 text-sm md:text-base"><div><p>Jan Hasenauer is a professor for computational biomedicine and leads the Interdisciplinary Research Unit Mathematics and Life Sciences at the University of Bonn and the Research Group for Data-driven Computational Modelling at the Helmholtz Zentrum München.   </p></div></div></div><div class="body-text mt-4"><div><p>The group of Jan Hasenauer consists of eleven PhD students and seven PostDocs, and his research focuses on the development of methods for data-driven modelling of biological processes. These methods enable model-based integration of different data sets, critical evaluation of available information, comparison of different biological hypotheses and tailor-made selection of future experiments. In various national and international collaborations these methods are currently employed to address research questions in molecular biology, immunology, neurobiology and epidemiology. </p></div></div></section><section class="mt-8"><div class="rounded-lg shadow-md bg-gray-100 px-2 py-2 lg:h-64 lg:pt-4"><div class="ml-2 sm:ml-0 flex items-center space-x-2 sm:justify-center"><div class="rounded-full h-16 w-16 sm:h-20 sm:w-20 lg:h-24 lg:w-24 overflow-hidden"><img alt="Torbjörn Nordling" src="data:image/svg+xml,%3csvg fill='none' viewBox='0 0 185 236' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'%3e%3cdefs%3e%3cfilter id='__svg-blur-64d75b7931fc2f9f486c8aa59e548ccc'%3e%3cfeGaussianBlur in='SourceGraphic' stdDeviation='40'/%3e%3c/filter%3e%3c/defs%3e%3cimage x='0' y='0' filter='url(%23__svg-blur-64d75b7931fc2f9f486c8aa59e548ccc)' width='185' height='236' xlink:href='data:image/png%3bbase64%2cUklGRt4CAABXRUJQVlA4INICAADQDgCdASpAAFIAPm0ulEakIqIhKVtrSIANiWUA0aGCSdbd43d5fdzXeLG1WrSLWC7gZuduJLXwK3F3n86mNF/59HgmyJcO7QVNpQDnyOGZ3UpVV7U3mfhzR27EYDQ6hZwemyfs4LOXcHCwkd%2bbwgE5b99NEkiaeksbN9GZrqIAAP78kdz%2b1u6fuX43wL0vNGOzcJURBO3xF/qasccnsJ7Pi7cGNr4/98q8l0iuaeKXpwRviaWw3XZ7w0LMvydXcfc8ufL3wIg1V00OXSlZNykTG7QemoxoxF8x9elyfXEvvGrUKSuYyqqwwUzLKIDrBU8IARCpf51IgaU7Yz5ajykYLIYXr3T9LwaJmJlm0toMvuTQpjzZqk8S3lpcHu2gcbRQoCtip%2b3r3xCsdaXvBDMpsNxnWFFwrONEQreZlnHkWNeQbRRX4UYg5JL/djVZWhK9K9b38%2bErvxqhHde0gQNoOiyevrD1c7eUNi8O/fMVaAySFe11v5q%2bKmCGRcqIW3FU9FdHYw2cZNaGWXeXGEiQX4PndLXUKpPnI0dA9kI2GnlmlKUKXVQ/uVC4OM2CwJA57RTPHNXDGSRkXVtOoZt7WGbQeMGzjqFP91Fe9qTB6cKtXaqDNY3Is1TuGoxUmvtHyT09uV7GcQwWnisiZqvw%2bv11z7n7umxrCamdUL6ydhp42L/%2bdb0o9JfxHxt%2b1cSV%2bYE164HrRPYkuRLwMXpt1wIccO8aNheulFwSYXJTMkLlUU1AwsUUM2EbGqGMY2RW5rajZ1X4UAajnG5moyXXTdJNOFtgrzl4o3OT3Qqhk0QIlbKqEvuWN6I/a5KqZr2GAazCcCC1Z954Ue5CN7xl5Uzi5PXQ5T3rNqpIlBEYuvk8VsHV2yANWn5hsxdvSASFDZYIBHo16ojjYLLhH1oG9h0sMTMYV6yJ6mU2%2bvTgtsiYxDTQDF3j00Ki8KwyAAAAAA==' /%3e%3c/svg%3e" width="185" data-src="/assets/static/torbjorn.690936a.52a10d3cad15196ef11d720b3449fdf2.png" data-srcset="/assets/static/torbjorn.690936a.52a10d3cad15196ef11d720b3449fdf2.png 185w" data-sizes="(max-width: 185px) 100vw, 185px" class="g-image g-image--lazy g-image--loading"><noscript><img src="/assets/static/torbjorn.690936a.52a10d3cad15196ef11d720b3449fdf2.png" class="g-image g-image--loaded" width="185" alt="Torbjörn Nordling"></noscript></div><h3 class="text-lg sm:text-xl lg:text-2xl font-medium">Torbjörn Nordling</h3></div><div class="body-text mt-2 text-gray-700 text-sm md:text-base"><div><p>Torbjörn Nordling is Assistant Professor at the Dept. of Mechanical Engineering and Joint Faculty of Miin Wu School of Computing at the National Cheng Kung University in Taiwan, and Senior Lecturer at the Department of Applied Physics and Electronics at Umeå University, Sweden.  </p></div></div></div><div class="body-text mt-4"><div><p>The research in the Nordling Lab is divided into two themes--AI &amp; Data Science and Systems &amp; Synthetic Biology. Within AI &amp; Data Science, their projects cover questions within Digital health, Industry 4.0, Internet of things, and Robotics. Within Systems &amp; Synthetic Biology, the research group focuses on Biomarker discovery, Experiment design, and Identification of gene regulatory networks. The core is a data centric approach to all problems applying existing or developing new methodology as needed. </p></div><div><p>In addition, Dr. Nordling is also an angel investor and advisor of several startups, including Darmiyan Inc.—-a MedTech company developing a virtual microscopy technique for early detection and monitoring of Alzheimer's disease; Virtual Internship Partners Ltd.—-a Digital work company that already is the global leader providing virtual internships to college students; Stakester Ltd.—-a Gaming company democratising e-sport, Trailze Ltd.—-a Navigation company for micro-mobility, Unity Chain, Inc.—-a Block chain venture studio creating the multi-helix technology powering Supra-Oracles, the flagship project of the Entropy Foundation. He has also co-founded several startups, such as ContentMap AB, Nordron AB, and Jagah Systems AB. He has given numerous popular lectures on disruptive innovation, finance, and entrepreneurship, as well as acted as a judge in startup competitions. </p></div></div></section><section class="mt-8"><div class="rounded-lg shadow-md bg-gray-100 px-2 py-2 lg:h-64 lg:pt-4"><div class="ml-2 sm:ml-0 flex items-center space-x-2 sm:justify-center"><div class="rounded-full h-16 w-16 sm:h-20 sm:w-20 lg:h-24 lg:w-24 overflow-hidden"><img alt="Matthias König" src="data:image/svg+xml,%3csvg fill='none' viewBox='0 0 200 201' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'%3e%3cdefs%3e%3cfilter id='__svg-blur-82383c7e5979f904b724b563e3ec9ce2'%3e%3cfeGaussianBlur in='SourceGraphic' stdDeviation='40'/%3e%3c/filter%3e%3c/defs%3e%3cimage x='0' y='0' filter='url(%23__svg-blur-82383c7e5979f904b724b563e3ec9ce2)' width='200' height='201' xlink:href='data:image/png%3bbase64%2ciVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAACXBIWXMAAC4jAAAuIwF4pT92AAAgAElEQVR42r17B1CU6bZtKyogGSSYMKfRMcyMOaKYI2YQAwoiAopgQEGCAipGEBPqmNOMadQZ0wSdMSI559hA524y0s2/7v7%2bBs%2bcenXqvXvuPY%2bqr/pXuujea6%2b19todBIL/wU%2bFTCyQqKT8tVgpEYgUkrayKpXO3%2b/T0KxqVyEXDxQpxQ6VssrgCpnkUblMnE3XKrFCohHLJZxUJa0TK8Sl5ZLKPyukldFCSZlLiahsdHF5ienf/1ZRRWGbwrJinaKyYv7fhcISQVFZkeD/%2b09pZZlArJLz15UKlUBaJdWRVynatP5eJBeaSVXyJRKV7IJEKcuk28%2b1TXVoBofWHzVd12s%2bQ1VfjaqGaijrlFDWKlDTWIPaz9UggFAuqagQioRPhKLSLcXlpQP%2b/hwKSot0igkA9lNGzyenIE9AgP1nCxerRHRk9OD52kKp4xXS8ratv6%2bUicfIq6ti5TWqynp1A5XIUVHVqKai6tW1mrySTPWvrx5rrt2Kbb7/6AYXl/Cay8pL44rL8zn6PaeGRlNVX62ukFSohaIyTqKUoJoHSIXiiuLPReUlLwrLSpyKhcV6rY%2bZX1Kok5GTxl/nFuYK8otz/zPFVyrEAqFUokVcKhVUykVfqE4UnipSSJ8ralWoU9dDopRCViXTNLKymqo1d%2b5f5lxcV3DDhw3iTM1MOEEbHXRo34GztDTHoEF9MPK7IZyz8yIci47gMvOSuEaujqtuqOYqpBXNldIKdbmkXC0Ul0FeIwdJDYVlRXmFpUVbxLJKHoiCsrI2OYV5fCPyigsF2QXZ1KT/RSAqZSIBaVire7n4Hx2XVw6qkIp/qvlMHdY0QFlfxaVkJqs/JrxrTs6I4/aF%2b3MzZ03iOnRox9Hd0VHPAMZGptDXN4JOu47Q1aVbnfbo0E4Xujrt0L6tDrp164INbs549vIhV9dUTSwQcRUSIVcuLuPKRCWaImExA4RAlqGgtDA3t6hgRevzIRl8aUpuUZ4gv0z4Py%2beTEng%2bAf4a6FU/OUBqBN76DQ1NH8mmiubyQfUMpWUyylIw9ZtG7levW25DlQUK7wdFduhgwEdfZgYGMLQwAh6uh3Rvr0e9PQN0FZHl/%2bdQUcj6LbTIyDa03VHBAT6opFrgEQhpuJLGSNQWlnCFZYVNtMhVpQznwBR/1FucWF39rwyc7N1cou0Es3KzxVk5uX8%2b8WLlFqHb2oUkwQk7di1TFXeXSgVvf5MNiatkkNWo1CLVRKuulGFV2%2beciNGDOQM6ckbUmFdLC1gbWEOYypat70u9HX1YaxPherpkwR0oUfH3MgYFiam0CUA2P3MjExgZGAMEyMzGNL9DhzYi%2boGBaRKMUSycsYCAqCIfKME%2bSUFGjpq8h5kF%2bQqs/Ozl7Dn%2bOqv16xw3pTTsjMFGbkF/0bnFVq9i2jMlUm0ei8Vi%2bzK5WJZLRmcpEreJK1SNKvqa0iXYu5c7BFuQL9ufMdtOpnD1toC3S3N0ItuLc1MYGliDDNDKpAAsDDQp8I7woSujfQYK/RhamDAA2FpakogGMPU2BRWnaxgamQEJ6fFKCrNJqbJiQGlKK0oJQCKubySfOQV53NkhE10UFJRhqz87LDWGtKys3i5pmRmCpLTk/4bmpdri5fWyAVEMR3t/4mcKuQSSGqqQMxoklbLISfTq2mq5x7/cgtjvhtCndWDNRVvqq8HCzqWHfXQiYrramaMId2tMKyHDWzp2pzuZ9GRHSpen4AgECwIHGsq3oodM3N06WRJoJmhe%2bfOPCtWLFuAzLxkqGhcVsoqQF6AvJICrqAknyO9g44mMy%2b7mSRL98u%2b1lpLalYmD0JSeqYgr7Dw/158Mc3UJkCg%2blxFmq/ki6fg4lrVSLOZ3F2slGloFEJRp%2bKUdJpICm4bHWGu3xadzUjDbdtCt01bWOh2QKcOdHTbw0qvA7ob6WN4104YaGmC/hYm6GVqDFtTE3QxNoIVgWRj2BEWerrEED3YmBihd9cu6GNriz7dusHGohP027XHtClj8Oqvp8gvygGTAI1D5JcWcHlFeQRCPskgh0vLTm8SikVIzc5%2b1FpTckYmL4fE9AxBfEbGvy6%2bnDpPum4Ze1rN0/hzUtAspplPndcWL6lScLJaJY2rJmQXJmPEsD7UaV10NTGAuW47WFLBvUwNMbxLJ4zvZYNJfTrjmy4WmNCrM6b27Qr7/raY2Kc7RnW3xihbK/p/G0zuZ4vJ/XtgWHdiiakRelqYoaeVBbqYm8KCZMDk0cnYEB7uq/Eh7g3rPigAsc4zGdBtLo3QbKTnZNFtblNBmRCJ6Wk/ttaWlJ7RRsuIrH9Be6XsH7pXSHS0UVc0uVJBXaeixSqpRlxFxVez4qn7BEo91wjPLeugS7rvbGyAnmYG%2bLqrOUb1tMa8Ef2wdfYY7F8xDTGui3DeyxF3/Nbhlo8zTm9YghCH6QhcOAUHls/EmfUOiN20HFGuS7B3xSysnPANvuvTg6RDxZM3GJNEzMlD2tPInDD%2bOzK/HJRLReQHZVR8AX/IBIn6uRzlAR6EuJS0pkJhBaj7J3gAMrPbZubkCt4kJggS0jL/TwDKJP885ytlkm6U9aVM50R7tUQlh7SleDpopGCbnBmHrwfZopNeOwywNMaIbmaYNaQ7NkwdjhDHqbjh64j3R7YiPWY3ii8EofzSfhSdC0HKMX98PLQTbyN88CnSF2/DvfHbXjc83e2C29uccXTNAqyzH0Mg2KIL8wxDA94rDClH6LVrh5MnI9n0IR8oQYGwiAcgqyAPmfk5yCnM5VKzM5GUns4lpGVQ%2bmRMyHBjNSWkpfOsjk9JFZBE/h5xFYLqenJ7SnoU01tGoOSPWk09CJAmyv2gPM/JCAB5bRVY9zV0x%2bPRYTDQEaBPJyPMHGqL1RMGwnf%2btzi0ehru7HJGYpQfxNfDIbu6H9JrByC5FonyixEoPR%2bBnJPByIrZi9TjO/AhwguvQ9zx3H8dHmxzQqzrQux2sMOKcUMxjqTRixKjpYkJLIyN%2bSmzdcsm/jkUlZeioKwIuTwD8ngWZJAMCAAuPjUV8WkZzcmZWQwMdUpmxlAtCGl8g//6%2bOlv2peIWotuMT3JrkY0gXygScSMT0Wdr1FxijpaWmjs1TTREkP5fOG8iTAmAOy%2b7gm3WcOwy2E0IlZOxPdeixEfvQPyWxGouRkB1bX9kFwOg/hKBCovEgtiApF73B%2bZx7Yj4YAn3oW64XWwG34NcMFDH0fc8FyCE2vnYueCidhg9w2mD%2b2PPl2syAMoM5DUDh4IgqxaSUZYCtoIeQAoEZIfMfpnI40OYwB1GtR1dWZ%2bIT4kpXxCqTYUvYtPFOQVFQmaWbNp62qhvqStNuHJBlDXG1nhBEAz675YpeQ7zwBQNdbiM3U/uzADg/vaYIC1KeaP7A33mV8jeOkoXPWYhzcHPFB%2bLQTVN0JRdSUYkvN7UBKzE/knd6HgpD%2byIrch67Afso74IZkA%2bCvQBS93rcEv251xz3s5rrjOxxmXuYhcaY8gh0lwnz4K35Jp2pAnmHXURUR4ID0XJXW/lFhQ9qX4rIJ8ZOTmEAOo6xkZ3MekFO5jcio%2bJKc15ZUKGRi7%2bKmQmcU3OjEtg3d7QYlQSIFH1KJ90f3qz3WgLKBmIJDzc2R8aAWAravs58HjW%2bhi0h7f9LLCnG96wGPWUNzathgpRzeh%2bNx2SC4FQHU5CPILe1Ea7Yusg5uQQJ1OCtuM5PDNSDu4BblHtyEjcgveBm7As22r8MB7JW5vWoLY1bMR7WiPqJV2OLR0MrbPm4Bpw/uij00nyhgdEBCwjZ6PkjyAMaCUGFBMABRSCCIA8vKo%2bEzSfSbiUwmE5DSuVQpxycnVbz8l2LI645JT2vwj9bUYH837KTTiSPeSZpFCxlH3ORHTP9GNJEAA1NBqW8cD8P2laHQ2aouRva3hMKYfthP9fwlahZIzPpBc3AkZASCJ3YXyk9tQeNQbGWEeiAtywzsq9v3eDUgm3ece24HcqO1IifCm/ycP2L4GD7asxHW3hThNEji8dArCHSbAf/54LBw5CF/R2LQx6YhdOzx5CZSKhMgnFuSWlPAAsOLTc/Oo%2bCwk0KHCEZdCUkhLA5sKWQXF%2bJSSeobVSkDo8H5HiU5QIipv0wLEL3z3FRJ1hZy0X0XdJ0BY8fK6KtJ9HapaADh9%2biC6m3TAlCG9sHB0X/jM%2bwa/ha1FxUU/VJzdBiEBURbjA9HZXZCeC0DFqd3Ip4JTDm5DfJgnad8beTF7UHouCPlRe5B1fDc%2bhvvgj2APPPHfgJtbHBGzbi7Cltph14IJcJ40FMN72qCbaUd4uK3mp0BxBQFQykYhY0ARD0JaTg4Vn4FPqRmM%2bqT9NHbLxiIZYzo%2bJqc0UPF9WL0fEokFlPbaaje9ipG004NYwJEZciQBjs19MkA%2b8rIwpKyvJQ/QAhAR7o8%2bFh0xY0QvzP%2b2O/wdviXtu6D4tCcdbwjP%2bkJxNQRND45D8zAGn%2b%2bdQv3D8xDdOIECmgLZp0JQdvkQah5dQPVPVyG8eQaJx4Lpb%2bzEixBvPNztjus%2bqxGzYRFCnGZhI2WKkX27oLu5IRyXzEGZuBLFlRUUiJgJliCnqIhOIVKycqjz6YijYj%2bkpPMgfKTimRfQUafnFeDNp8RIVvOb%2bAQdWnC0cZc2vNO16kbqvlRNCxDdarsv4Tc%2bLf1Z96s%2b1/MvbO3yc6XUZop5o3tjzZQ%2buOy7AGknNyMrahOEF3ZAdiMMDT/FAL9dBV5cAffkCjS//oi65z9A8fAyRHfOQ/X4Bmqe3IDswVXknT9K2SAAf0bswtPQbbgf4E3TZA3CKBgFLLOH7/IZmDa0D2wtDLBsgT1KKoUkATHRv1UCxICCAjLAHK0EMpgEMnkQ%2bOKTUpkfNDNviEtKFSalpxr/Y%2b2VVZrQ6CuXUqEUfZspEVL3%2beDDa01araLia6n4Brr9TAMS8N64DOP7W2P5xH7wcxiBV8c2IuO0F7JP%2byCV5n/B9/sgvRsD%2bf0zEN2KRs6pfSg8exCF5yNRcPoABaMoVFw/hawTwUg4tIuygD/ubHXF/uVzsXXuNGyZNw2bZk3B%2bmnjsPjbQXCy%2bw4rpo5EP2sTOMyahILSIpSKxWCRN7uomCZAER1iAAFAqY8H4RMBQEWTHNK1hkggxCWnalJz8hgLlv8jBUrFi1QNtWR%2bUg1zfVp6SAaM/tR9HgSiP%2blf1VCPOo2aInATNrksxOQhXbBkXG8c2TgDSWe98eaQK274LKHYuxxnPJfjMEVc/0VT4Dt7Ik6uX47bPhvx83ZXGn1%2bKD65j7QfSiPQA4991%2bLKJidy%2b2mwo%2bAzjNx%2b0oBecBg1DI4TR8JuYG/YfdULy6aNwuDunTDXbhQVm0tylVESFBL1S5BVWIxMAiA1JxdJWZQDMrIJgAy8S2LU5wGgKUBekJSqTskpwLv4pNt/e3lbdK6auksAqJn7iyj7Vypo/ldR92uU5AEMgBoegHqumX/5a%2bOauZj8dRcsHtcHF/0ckBC1kcLLNOyYPxaR6xdgP40yl0nDsfTb/nD8pj92zxlHzj4Pj7Y6ovDEbtRcOwphzH786uuKyy4O2DtvPNaPGwKnUUNgP7gvnd5YRs6/bsLX2DxzLJynfIel08bgK1qe5k8bTSMvjwBQEABlvAcwh0/PLSATzCcfoFGYSV7As4DGXwqZIkmBGWFcUlozuyYAyuOTU00EcrlEh9w/XUZznqivoQlABihj%2bR%2b8DAgAGQOgvo6fAvXNjAGfscXNAfbDu8NhbC9c2r4Ib49uwIn1M3DFdyVu%2bK3AHofxmDmsH/qY0y4/eRz2Oy1AlJM9XgaSg9NegIcnIYsNxYtt67Bv3mS4TxqGgAV2iNyzE327dQUZE2aMHYnxfXpixTBb7HOaAY9lszGkZ2csIkDySooglEopBwjpuoxYUIw0AoCZXGp2LhKJAQyEBOYDyen8%2bZhEABAL3iemNMelZuJtQtIUQbFI2p/038he7GDuTywgAOQ8/ZkJSqu1DJDXkQc01vM%2bwEwwItgbdkNo6xvZE1d2LsbHE%2b54HOSMnwOXYfVgC7jMtcdC%2b7EwpUJ2OC3HTQovtzcRU054QHzeH%2bqbtBvEBuO53xoEzBoPn9mjcWzNPFz03wLvuXaY93Uf%2bMyZBq8pY7BmZH/4LxiNENelGEexeN3KuWSC5aAFjpdAbgljQSktQ0UEQj7FYcoCmS0AsEDEWEBeEJeczueCD4lp6rS8Irz9lLSLuf9C5vAEgKZCQcUr%2bPRHh4qv0hqgrLaGxiCFoKYGHgD2c//eeUpnXTDr2x644b8cOed88PHYegoyM3DKcSKe7XXFj7tccNXHBY92b8LPNNbeHvFB3rmtqLrgj9rLoRCf3o3fdqzF/kV2CKTcf9BxBs5tWE4rM%2bUAn7U47jgd0c4zcX7zEhxyno7IjUuwdOpYuDvOh5DGYKlESlG4kgeAbX1MBowFqdl5tKlqfSCRP1lsCvBSYCx4n5CiTsnOJyNMusVe/gpkhVWS/okJBICE77xYqWWArLqKxqCWAcwnqmkMMghSE37F/rV2WDi2Hx6FrEXZ1V3IjfVE1knq8M1ACCkNFp7biUwKQPEnAsgj/FF4ZgdUFwNQeyUUklP%2bEB3Zidd%2ba3FoyVSE0fZ30nkubng746YvW4tX4Qc/uqbt8HuPxTi3eSmOb1oG90XTsHbOZBSXlqJMSitxeTmlQWaEZcSAYjCHZyxIzsrVMqBlHH5KyeAZ8CExlfuQlKqJT8/GX3GJn2gXkD5g2qbdX1PJ6K%2bQ8QyQUAiqVGozgHYFZjmAACAZsChUGvcL7rnPhOu8UXh3xJOWH9r5r%2b6E5M5eiG/RDnB7H0TXg1BycS%2bKzgVCdH4v6q5HQH37EFSXQiA9HYiamBDE7diIqBVzEbJwMk6vnksMcsYvO9bhCTHjMS1Hj3aspvHohEtejojyWAEfygVei2ejiGY/A4C96ME8ILOATYESvnh2%2bM6nMxnkagFI1Y7EDwkpBEBaM5sMxAAhxV9ZJtO5UCpprmTGp5S3vAJEIYjRnwCQVle1pECSAE2CRgJATAx477sMQSsnU3d9ILwSANFN2gB/jIDyeiiUtAVKY3dDcoEtRGFQXjiIph%2bPgvvxECRnAiA%2buRc1p0OR6O%2bOs86LsXPGBJxZPQ8Pt67Ci10b8SrQE7/u2Ugrsjue7HTF9a1rEO2%2bHL7LZiDY04WSoJhygJwYICIJCJFZWIJ08oAMOsnZeXwWYAxIysjhPeAjC0RaCRALmBGmMQ%2bopgVIqmARWCiT8gbIABDxxbMUqNSCQDlAUa81wVYJVBWlo%2biYJ27QBpgUsxXF3%2b%2bB5MY%2bND06Ae7xSah/OIb6a4dRfZXOpSOou3aMnD8KiotByAr1RMXJPdDciUJZdAjuejiSB9hjH02BXwM34m2oN14HeeOvfVvxMsADP%2b3YQAA4IdptMXyX0P383FAiElHTFMgnBuSWlFMOKCEQivkpkEz6TmTFZ%2bXwIMSlUPpLzeBlwFjwjljwJj6FhaEm9n4faV/KMgABIOfNj%2b8%2bO8wDiP7aMahlAPMLFoalciFKLu/DK1px46K3oPAC0f/mfmieRFP0PU8gnIPm/mk03YpCw7UTaLp3GvUESsnxnag%2bH4b62wcpJl%2bH%2bvdHeBfmi6Or5uPQ4lm4v3UdsmLC8X7/DrwP98PvtBc8IAAue61ElOsi%2bFCwignZQc%2bzilggQ56wkjygkp8ATAbM3FjQScpmkyCXB4IV/5HPARlgnX8Xn8K9jU9mEuBIAnIqnAUfSYv%2bFfz6y1jA05/fBGvp1PFZgE0BxgBJtYTibiTeRvji7QlPyvJ%2bkP4QjqbHUeCenQP3/BIBcQnN985AQxOj7lYMKqnwilN7UUameH71HBSfDYLw%2b0M4TsVvsR%2bPaBpvV9Yuoc0wCOknQ/GRFqNfg7bgnt96XPBageOuC%2bA9fwJiDwZDVtdAEpAhv1yMvLIKZBSUaGWQx0ZhIZlgHl98AnkBH4haQGAAvCcjpEnATJBjEmhgAGgnAIvBTAKMCUT/Ku0EYCxgICjqmQQaKQxpoCImpJwOwTMfR/we6Y7sWF9Ib4eh8TFjQCy4n8/zIHC/XKaN8AIqz%2byH6PQ%2blB7bjVe%2bTrjqsgh/BbnhxHJ7bJ06BufXLEb0ktl4uMkZH4M2IvnwTnyK3InfQ7fgrp8LYlm03jAPnpQYLxwNp2TaSABQEKpgWaASWTQFmP5T2RjM0Z4kmgTxPADZvA98SGIA8CAQAKn482OChr0aLKyQKwgAOVcul2lloFLyR0wgVChYGKrhZSBjINADq2ghqmpuRv7P3yMnehteRGxA%2bjlfyO%2bGo%2b4hGd3L8wTCZQLhIknhPGquH4X4bDDqroSh9tw%2blBzzRyIV%2bJQM7pqXM6W8RdgwZjhC59jRZFmGx5sX403AenwI8yIGeODuzrU457kMEetmw33WWJwK2gk5NaNMQlOgQkQSqEA6dT%2bND0IUh3kfaGUAk0Amf5gJskRIxbNJwDyggTHgnaK2lv6YpFkoYwxgrwjRUShpDLKjQgUdMYEgqanlqVfV2Aglx6H47WNILwfij0NuSCAjVNxjABxB05MYksFF/jTdO4na28dRSyOw/so%2bKCn%2bsqK2kOFtd5iBqPUr4TN/Gkb17IoFwwdg74LJiGEBaMUU/LRtBZ7tWY/bfqtwatMihKyyx/rpI3H5WAQPgFCm4BmQW1qO7OIyvvsZLAzlFfLdZzJgU4CNQCYB5gEsCTITZNd/fUqWMRM8o%2bTRlKgpAnNsCWJHpGwBQMUAqIKoupY/Yh4E2gvIB/LfPkP1hb1IidmGP4%2b509obgpqHkai9exiNd09C84hM8N5RWnzCUXSGBaFAvNrngegNCxGwYjrCVs7GTc%2b1uLvdBcGL7bGRYq/zhKEIWDgRrpNGIHDuGPzktxJXvSkEuc3DnhV2FLHH4vKlKJJAEyVBOQrIA/JJAkz/LN6yUchMUDsJcrXFp7UAwJiQksGCUDPzhrefknNJAhI3FelaKJWpKxWKlkkgh1YK1XSqyPBqqPgaHgBJbR2kBEAtAZCSnoC/TvgjPopkQKtwwaXdqLp3AHUPItHw8Dga7h9H8/2jSD3shUdE6TvbV1OyW4XnwTTqwryRdmA7ciIDkRDoTbRfgZhVs3Ccou8R59lYP3UUltIWeZEKv%2bSxCIdcZmPniqlYt3AcTkYFkyc1UBCSo7hSwo/CdLYNsiBEaTCFTDCRup/A7wJaD2BbYcvrA9yH5DQNYwvlgJe0CkvHs3lPuzXHDtGfWKAAk4K4SgUhW4woBzAmMBBaAVA2aZBZko/Ig744FLQJ9w56IJN8gIWhqh8PoPZeJJoeHoPqZih%2b2O6IaFqE7geSrg94Iy5kI/6kpPdikyMer12IGzTbz8yjXWDGWARO/RZ7507A1lmj4TxxMI6xXcB9PvY7T8PWpZOxif796s%2bnUJIPlUlkBIBWAhmFpXRKeCBSaCX%2bMgYztCAwBnziN0MWglL5ZYgmQRRFYbkFFS4Rkc7L5fJmxgIxM0Ge/gSMnMyQpgEDgR1JTR1JgLZCdTPKZGJEkaGFBm/GiSBXxJEPlFzZAymBoLoZDvXjYyiI3Y5Da6fRCJuHa7RCn6RCL1Oa%2b9nVAT8unYtrC2biruMSXJhnj4jpE%2bE1%2blsEz5qMs27zcYCKvriJpsP62QhaNQ1u88cg9gxJrJkjj1KRBGQoIgCYCWaRBzAZsM4yejMQmAHGt7xCHMdkkJbFvadliAxQnUwyeZ%2bctrrlfUHpC2VDAxUr1VDBHMsCjAWM/lIagwwAUVU1f6RUvJxMUE7TQPFZjffxrxETtQdB/i54cHATcs5vh/AaRd1rQai6exDFlwJwZvMcXNy6GJ/Ct%2bJPmukf/NYhO8gLOYE7kBuyB8mB/njmsgYxlAZP0Cj83mU%2bLUVLcc59AS5sdkDE6unY7Uj0nzca93%2b6hVpNCwC8BMTIZlG4iBjA6M%2bPvzw%2bEDEZMPozQD7RUsS6T6OQ4ydBYtpnAkD70Tsagbtq1Gr6o3LyASUzQo4xQkI7QDkLSXRdybOghhajehqFDQTAZ8jIiOrJC9ibpbd%2biEXkDie8OepJW%2bF2lF4NQNm1vRDe2ocn4evxdN8GNP10lBJgOAWhXSigAJUXuh2pe/zwwt0FN1ctx6XVDrjk7oDLmxfhKrl%2bpDPF3pV2CHWyx%2baFE7B%2b%2bVS8%2b/gadc3gk2BJyzqcQ8sQywHp/AQo0mYAtg9k5/IsaGXAe5oANAU0SVn5eJeYGv%2b3zwWIh5ZJpc0EAJMBxw6TADtlMnlL94kNpH8xSUBe3wAJMUHxuYkCEfuIBBlibhpOhXngZqAz4o57IeP0NuSSKabE7kCM1zw8D3eD6gYtSdeCIbkWgpxTO/GCZHOMEuHRdQtwxXslbmxdgR/J9W95OyB2w2zsd5qKLXNGwnfROCydMhxbNi1BTmEWpdEmMm05m1w0BSr5wzyAN0JiAWMAOywEfckBaZkcH4mTM0j/xXibkLxP%2bzGYBlWLDGRvVERtygIaBgBLhJUto5APRlXMBAkIGoOseFk9BSK1hpdBDWlSSiHp%2bQ9RuOC3ALG%2bS/CWxmLOBT%2baEJ44uNYO90PXopQyQ9X1faiinaGCtsXUk774LdQDf4RtxoeDtACFuuLxjhW45jEPR9fOoNQ3BmumDMWyiV9j5oQhiDzgy6dTOT1%2bJUmghE%2bCtA3SFGAhKJW6n0zaZyOQAcCKZ0YYT7cf2avCKZnMAMF84H1yxvAW%2bsta3hGWu9c0qYkBCrVImwA5pjPmBWKeATXaMFTLAGgg%2brNDINA0qNU08yty4oenuBeyHEGrxyHSdTp%2bCqCYfHAdznovxPW9axB32ocWqL1ouBWBxjuRUN06jMrLh5B3JpjygRcVv4Zy/0oqfjr8F4/DkrGDMLqvDQZ2s8KSOaPw4OFl/vEkNI6FMiWKRNoorPWAMh4ALQhkhPwixIi%2bQAUAAA04SURBVDyAglBaNpkf/7qgJoVMkgD4nX977ENSG5YE27R8HtCUskA5c3pajZv5JUmh/GJ%2b/OHTIE0BCk5SXgJqVDEAiAkMgCJakV8e34Dvd83HuukDsXXBcAo8Y%2bj6a2xfOQXX/J3w/thWVFwMpawQhcrbJ5B35QSyvj%2bMh0GbsY/2Ap85Y7GOuu44fiDGUPFWhnroZWWC7R5LqbOpqCEDZPlEKNcCkC8U8QCwKMyKZxkgqSUEJWTktgCQxX1MzSTzy9Cw%2b5AB8u8JvE9O12l5X0DOX5SIZUH8AyhVTazzFcwQaRKwacAAYFFYSt1X0FrM5rCCTJABUEN7AdsQK0UleH3KE08OOCPAaQyWjOuOyQMtMH6ABRZPHIB962figuci3N69Fvf3e%2bFh2Ba8OOCHH/e4IXjVHAo/I2H3lS0mDuiG4T2s0FGnLXTbtsHM4b1wLiqUH7/KxiZiZA3PgEICgK3D2cVCpOZrDZB1P7ElAzAWfErP5tgkiEvL0iQQI4gFKYpmGf92IJmjQKCskQuKxVoWFFZUmBGylaxQyv/NIgo/RH%2bOH381tbwBMgBY8XLyAHZYHmBewKaBvEqOd7QWvyIWXPBfBM9FwzCqtwmsDdqgi3EHjB/UHXudpsF58hCsoi5vdZgM/2X22L7YDovHDqWOd0c3M0PYmhujQzvtp0y/7mEJ15kj8PPjH/iX4qT0HERVtLvIyQMoCueXiygDMAaUgplbMj8B8vnzKZ0ZIAMghwrPUCdlFZAJZjryb4y2dr9UrP1gFIWKVi/wrCZNU%2bfVYm3neQBYHBbz9G/gM4CML14DJZ1qMsFaDsSIOny4EkCByB1Pj7ggbP0ETBlsgT6ddGFt2BYWem3R18oQg207wX5YT0wa2BWj%2bnejIm0wlI61sSGoNXzh7FgZ6WGV3VBsnDUMr/94zr8aLa1rRLmyGuWqGhTxL4iQBErJBBkABQwAFoPJA8gEtd3PYbfq5OxC0n7Ga1bj/Sd/CD6l5QhoL2j5YLGwUkBh58tYJBa8qyJDpO6r2fIjrq7ltHsAuX%2bDtvv8LdGxmiRQTRmCeQB7vyDuzgHaDN0Qd44ofmgNvBd/QyBYYtIQSwzuZgCT9gK0o%2bL027aFYft2aC9owxfbXkeHbtugDftsMVHf0rADFo8bQCwZD6/5IzF1/DicunADdfQg8kYNStgLopUkAVqGcspoDBaVkwxKtADk8ABw8eQBnzJzuQ8pmXjHXgtISvuW135SOi8BkkbrByS0nw0skyn4XxC1hlH40Yj5DbC6mXWeDse7f/1nKMn85PytNgOwHwbKiXMXEbl5IXIubUXWtZ1Iu%2b6Pu%2bHO8Fs5CttXjMb6mUMwf0xPjO5vgYFdjNDfxhD9OhujLx0rEz2Y6umgh7kB%2blqbYvqI3ti9yg5BztNpdR6DyV91IYDawnHNBryNT0G1hjynuh55tAxllZTTFCgnBpQihWSQlF3AJdKJJybEpec0ZRZX4GNaVmhL8TpaCWSSB%2bS1BiEFf/juK5T8R8mocG%2bmOWKAWlrbwInpkP4JBG0EZoVTICOnzURAUCiGfzcWvSxNcMV3DpJivZF7czcK74bg/QUf3ApfjVh/B5zZsQD7Nk7HrtVTEO45Hzto1ns52WH9wjFYMGEwxg%2b2xcxvesPJfjh20OZ3dNNC7KeRuHHWt/CYPQJTBjEQ2sHMogvcvShDvItHiawaZYo65AjFNAGo%2bJxCLim3iGNAJGblN2UUl1Px2a9b2Z2Yky/Y5bFXQBnhnz8nKKmuFsRnFAiU6kaBSFHHmyJR/gorUlJT30SHUOc4YiBXRZPitzfvsWa9G4xNLKBvYApTI2NEOI3Ei4iV%2bBCzGQW3AlF6fz8K7%2b1Hxp0gvDjpiadRG/nbKyFrccxvKSJ9VuCQ9zIEuMyF1zI7eDlMQuDqmTjgvgintq1EjPdy7F83kwAYAZdpQ7FqQh%2bYGFtA18CSl41%2bR1PMnOOAmNhriM8sRJFExRWKVVxqoZAmQaE6Nb%2bUxmC%2b6FNaYWdWz6d07ZcqaCL8i4/Hq7Q%2bIG9o4AGg2tvI6hpfMYoT4z6zsRMTe5GbZj%2bbM6HC2V1MTK3QuVtvzP/GFg/8Z%2bNx0GK8j96Ewjt7UX4/DOKfj6L88WGkXKO9/9IuJF3ZjT/O%2buJpjC9%2bjtqGW5QCz%2b7egJidq3HScwVOeC3H%2bV0uuLhjNU55LUYIbYGb534H91nD4Tl7MDpbdEIHAysCvRM9vq7WMNvoot%2bAodx69y3c1Vs/ESsLNAUiBUgCTUnZRaP44jPydbS3OYKE1Ox/8RF5AkBUU81fKxoa27aAYPj7uw8pHt4%2b6NW73%2bc27MsNukbooGfMGRiZw6pzD/TqbIXdC4bil6CFeL5/Cd5FuaPw9l5U/BROAERC8vQ4yp8cQ8b1AGTcCEYGLURJ18Pw5lwAfjnmhx8ituBGiAcuB2zA5cANuB7ohsv%2bqxHt6YAA2gL9HMZh28KRCFg2Er1trCHoYAa9jp3QVscA7fXM6LmYc4I2ehyTh76%2bqXrQ4G/gtc2fAlDeHFbDz7%2b942X9NjlH8PFfdf%2bLFOoaBMrGRv76xJmzrd8Q6dRe1yBFV98YpuZWTR0NTLn2HYxY9zkzq%2b7cuIG2OOA4Gr%2bFLcdfhx3xKcYDeTcCIPr5IOQvo6D4NQby30%2bj7NFh5P0QjpwfIpF6%2bxD5QzCen9iJe5Fb8WOEN24Eb8LVYFfcDnHFFf91OOI2nwKVHUJW2yN01RTsXjoaNhbWaKdvgfa6JgSAIdq07ci1bWfAtdczpX93bGKsaNOWZ4YDe%2bL%2bweF88X%2b8SxGk5f0/fqtM%2bblZUNfykdn1Gz14EE6du2DQuUuPlwZGFoS4kaaDnqmmk1U3GJl15iYN6cNFb5iKVwedEH9qA9Iu%2bSLr%2bm6UPgiD4mU0al6dQ82fFyAjIEofHUXu3cNIvhqGt%2beD8fLkHjw6uh0/hntT4Ztw94AXTQ4PnPNzwkHXeQhePR2HNi7Aae8FWDj6K7Rh3ScPoIK5jkadOCNTG3Q07ERs7NSkZ2BB1xZVPXsNmMyes6fv7i9f8UnOLf7vfWPk5e%2bvBeaWNvz1oMHDvvwhK2vbaGNTaxibd2Zu3ETXXGerLghaOYV7HubEZVzeitw7u5F7ay8KfwhBxZNIYsAZVP9xnm5PQ/zsJMp%2biUbBg%2bNIvBmJ1%2bdD8OzELjyM9MHDIz54cmQbbu9z503w4MZFFKbm4KyfIw5sXAxzetw27U3IAyy5dh1MOV19MxAAGgMjq2YCAZY2PZPGTrLvy56nvqF5u9bnPG/%2bkn/vO0NLlq4UdLLqyl/3GzDky7fFBg8dtaJz195yA3pQU3ObZt2OFuqv%2b/TlLvguo87v5PLv7eWKHuxDyf1wlD04AMnzaCh/Pwflb6chehYNIQPg4QkkXDuAv86H4tfo3bwXvIzxx9NjvrgetAHR25xw1HMpYrbSNPBxxtf9%2bnNtdM2o69acXkdL6LQ3atZpZ9TUtp0RycGY5Gh96uihi%2b3Z8zM0tvxS/IgRI/9n3xqbNm26oHuPfvz1dyPHsenAAzFmrF1nc8tut9hMNjWzgb6RlWbEwK/U10LcuKJHYSh7HMGV/nSQK314iKugSSAjFij/iCUwTqH0yQnk3juCJJLBe2LAHzF78OfZILyJDcIvx33xPek/2scJV4JcEeO3lhs56CuuTQdzegxrztC4s0ZX31zNPKCjkRVMzDpndOveZ2br8%2b3Wo38LW9sJxoy3%2b9/53qDn5q2CEd%2bOEbRMBYFN195fJNG3/zD7Ll16v7Gy6UF07ALbrr0R6LpEnXtvv6b%2b3Rmu8vkJruRRJCd9fpKTPqPz61nygSgU3D9K%2beAQPl0Ow7vvwxF3PZL2iHA8jvThLvq7cDdCPbgjW5y5If0GcB0MrJs7GndWt9czZ8WjAx0jY%2bvK7j2%2b8kvJLW/f8lR0hgwdxY9vC0tbwYJFy/93vznq6ekjmGQ3S6DX0Zz/d/%2bBw76wgf0M%2bGrEvG7d%2bjwzMetCNLWB/chvcJ9oXf3usrrh/UV1/dsLmpq/LjZLX13kyp6c4goeHucy7kRyidcOcQnXD3MZ96K47AfR3O8n/bkzO1ya186fqba27KHWpeKp69AztOLNz8TUJtvCsvuOKXYzzFof28Ky25eGDBw0VLBs5Zr/3PeHHVetJT/gv3sg8Nm%2bU9DNtv8/fVv8u%2b/GD%2bvdd3C4oWnXVHOL7pg7xQ7R/u6IuxEJ6e8XUfv%2bBuo/3uKq/rrCiZ6f48qenufKfonlSp5f4F6cDsEmh7mw7dITbTtYwMCETZluMDHvVmFu2eN6z75D5t%2b887C14wIj0646o8dO5a9tuvSlrq8QvHz%2b9j//7XEmg/XrNwp69xnU4hOzBLa9BjIg2vztPm2/Hj5qRAdDG29d/U43bLv2SZkyeox8zYK5ml1ua7DPaz32e23Arg2rsW7RAoz/djSsrXpy7fUta/QMrAssrHo8te7ab79tz8Ez7Gct%2bqev0esZdNLp13/Il8eaOGWaYOUqt3%2brlv8CBrbrIC0%2b/AUAAAAASUVORK5CYII=' /%3e%3c/svg%3e" width="200" data-src="/assets/static/matthias.7ba8522.dc139f940d8164bfec21d08fde81e2ed.png" data-srcset="/assets/static/matthias.7ba8522.dc139f940d8164bfec21d08fde81e2ed.png 200w" data-sizes="(max-width: 200px) 100vw, 200px" class="g-image g-image--lazy g-image--loading"><noscript><img src="/assets/static/matthias.7ba8522.dc139f940d8164bfec21d08fde81e2ed.png" class="g-image g-image--loaded" width="200" alt="Matthias König"></noscript></div><h3 class="text-lg sm:text-xl lg:text-2xl font-medium">Matthias König</h3></div><div class="body-text mt-2 text-gray-700 text-sm md:text-base"><div><p>Matthias König leads the junior research group of Systems Biology and Medicine of the Liver at the Humboldt-University Berlin.</p></div></div></div><div class="body-text mt-4"><div><p>König's research focus is predictive modeling of metabolism, including projects such as personalized evaluation of liver function tests, multi-scale modeling of the liver, and databases such as the tellurium-web for reproducible simulation experiments. The research group uses both computational modeling, data science, bioinformatics, and machine learning to understand biomedical systems. Köning has also been involved in the development of SBML and SED-ML. </p></div></div></section><section class="mt-8"><div class="rounded-lg shadow-md bg-gray-100 px-2 py-2 lg:h-64 lg:pt-4"><div class="ml-2 sm:ml-0 flex items-center space-x-2 sm:justify-center"><div class="rounded-full h-16 w-16 sm:h-20 sm:w-20 lg:h-24 lg:w-24 overflow-hidden"><img alt="Andreas Hellander" src="data:image/svg+xml,%3csvg fill='none' viewBox='0 0 699 647' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'%3e%3cdefs%3e%3cfilter id='__svg-blur-367404b4de12982dc35f4ab988a828cf'%3e%3cfeGaussianBlur in='SourceGraphic' stdDeviation='40'/%3e%3c/filter%3e%3c/defs%3e%3cimage x='0' y='0' filter='url(%23__svg-blur-367404b4de12982dc35f4ab988a828cf)' width='699' height='647' xlink:href='data:image/jpeg%3bbase64%2c/9j/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/2wBDAQcHBwoIChMKChMoGhYaKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCj/wAARCAA7AEADASIAAhEBAxEB/8QAHAAAAwEAAgMAAAAAAAAAAAAABQYHCAMEAAEC/8QAPRAAAQMDAwEFBAYHCQAAAAAAAQIDBAAFEQYSITEHEyJBUTJhcYEIFUKRocEjJENysdHSUoOSk6Kys8Lw/8QAGAEAAwEBAAAAAAAAAAAAAAAAAgMEAQD/xAAeEQADAAICAwEAAAAAAAAAAAAAAQIDERIhEyIxQf/aAAwDAQACEQMRAD8ABxYWlnyAmW%2bvOOij/TTAxpOwEAkST/en%2bmvNGaSbmN96VbdqW%2bPlVRGnGQAQcDHSsjJVbbY16ROE6S09jlh8/F1X8q9q0tptCSpUReAMkqcXVKGnmvU0sa90TMu1uQ1a5KGsbt6VgkK44rLpyt7NlKnpCItjQ5dU2G2itJwcrcx9%2b7FcqbDpl9vvI1uZcR03JKlD/fUlv1uuFkfVElBwLYyk7gR8CD5g0c7HruhvWUa3znlIjSwWfErjfjKfxGPnSldUtphuVL00OEqwWRIJTamfm2T/ANqDyrXaWzxao/8AkD%2bdW1/TcZXUGhE3S0Ug%2bE1u3%2bg%2boR0CztiK/ca/hVCDY2jFJGh0/qh/cb/gafEgbBxR4vguvpxKQKWddLjM2dK5aHFAuBCO7d7sgqBGc5HGDTWQDU%2b7TIwiWS8TZ90dU1IS01Ci92kBlwf2TkFW44znoK3LO5aQWKlNpsnet7bbo9kfhNPOPulO1p6S5uIOeOfSgvZBo1iTrZDjkJ12JaWgpcpSClDkndkAZ64B/CgDcCcjbKvUxL0Zsd4lppRXvPkD6c8Yp57KO0Ru3SX7XfgpqLKkFyPIPsMqPBSr0SSBz5HPlScOKktsoz55vpItjjddGQzkciiqsEAjkHkEV1nk5BpjRMI1u1DGsFuZclFKe9QnapxW1Awecq8jg8Z9KpFnukO7Q0SIDvetKAIOMZB6GoXLbj3SJChurfJktllLbZAyc5BBPAIOMZBBpdn9pUvS1zhW2wpdYVbEGLKRLIX3xSfZVjjA5PGOtLxVXLQ64jhyT7NSVItRMo1xriSy/uXZLH%2bh2gkB6SRk/JPB%2bIFdaF252yXpee%2b%2bwuHd2mFqaaHjbWccEK8uucH0oey8uxaDiREKT9dy2y4lS17d7r3iV4vXnH3VQ3omSb6QnX6NAiy1W21OuqgsEgqcXuJP2jnz8/8AxpUu1wacZSI6UqSeh25wPKqXL1npHT2iFQnrY4q7S4qmpcZDZDqVkEHctXQZ5GM%2bVRKK6g29hbilpJThBUPCfn5Ua0vhzT/TQnYJq5%2b6RX7BPVvchNByM4TyW84KD64yMe4%2b6qs90NZx%2bjssnXUtYwGm4K9xUcYypIH41od99GDhaP8AEKC9bCXwg%2bnZubtYdx9l78xTfrbsVsWpJT0%2b2yXLPcHlFay2O8ZcUTkkoJBBPuIqb2NR%2bsbNz%2b2/OtCMrVtTyetTJ8X0VRKpdmX7/wBmeotK3i2Q5y4siHcJbcVmQy5lKjncQpJ5HANNuv8AUtumXFVqfbRKiMFLjXeEtlHHBQodMDj06cU/9qPin6TJ5KZzqh7iGFc1nztKUV6xn7jnwtj/AECm8fIhXLw3pdhHtK1NA1ALehpruHGEd2Vbys7BgAEnr60pQEpYYWkTEbVZOzZwQR9rNLhUrdySfjX2latiuTyaZEqVoXkt5KdM0H9HJuMImpJLqEr8bLSCsZwMKJH4j7hRLVV%2beZeCIi0Iyog%2bBJ/Kl7sFWpOjbwQSCZw/400P1AtRmDKj7ZrrepGzPqj/2Q==' /%3e%3c/svg%3e" width="699" data-src="/assets/static/andreas.d565fee.1d8da286713d1b40138a3779f961a1a1.jpg" data-srcset="/assets/static/andreas.d565fee.1d8da286713d1b40138a3779f961a1a1.jpg 699w" data-sizes="(max-width: 699px) 100vw, 699px" class="g-image g-image--lazy g-image--loading"><noscript><img src="/assets/static/andreas.d565fee.1d8da286713d1b40138a3779f961a1a1.jpg" class="g-image g-image--loaded" width="699" alt="Andreas Hellander"></noscript></div><h3 class="text-lg sm:text-xl lg:text-2xl font-medium">Andreas Hellander</h3></div><div class="body-text mt-2 text-gray-700 text-sm md:text-base"><div><p>Andreas Hellander is an Associate Professor in the Integrative Scalable Computing Laboratory at Uppsala University. His group develops theory, methods and software to facilitate scalable digital experiments of complex systems. As a part of this, his group builds tools to simulate spatial stochastic biochemical reaction networks to study complex biological systems.  </p></div></div></div><div class="body-text mt-4"><div><p>Hellander has as a part of his work created the software StochSS for modelling and analysing biological system. He also works with machine learning approaches and is co-founder and CSO of Scaleout, which is a platform for federated learning.    </p></div></div></section></div><section class="mt-4 pb-4 border-b border-gray-400"></section><section class="mt-4 pb-4"><div id="Keynote talks" class="h-0" style="bottom: 100px; position: relative;"></div><h2 class="section-header">Speaker presentations</h2><div class="lg:grid lg:grid-flow-row lg:grid-cols-2 lg:gap-x-24"><section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Maria Kjellsson</span><h3></h3><span class="italic">Multistate Models to Predict Disease Progression of Type 2 Diabetes</span></h3></div></section><div class="body-text mt-4"><div><p>Drug development for therapeutic areas of progressive diseases is challenging as the natural changes in the disease over time needs to be accounted for in order to generate reasonable predictions the expected gained with a particular pharmacological intervention. Type 2 diabetes is commonly described as a progressive disease where the progression starts before the disease manifests with loss of glycemic control. In this presentation, I will describe how we have used multistate models of sometimes competing events to develop predictive models of diabetes onset as well a progression of long-term complications. </p></div></div></section><section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Jan Hasenauer</span><h3></h3><span class="italic">How can mechanistic modelling profit from machine learning?</span></h3></div></section><div class="body-text mt-4"><div><p>Mechanistic mathematical models are powerful tools in modern life sciences. Similar to experimental techniques, mechanistic models enable the evaluation of biological processes and hypothesis tests. Furthermore, mechanistic models allow the integrative evaluation of several data sets as well as the prediction of latent variables and future experiments. Yet, the scalability and need for large amounts of quantitative data has been encountered as a bottlenecks?  </p></div><div><p>In this talk, I will outline how methods pioneered in the field of machine learning can assist with the development of large-scale mechanistic models. First, I provide an overview about the interfaces of mechanistic modelling and machine learning. Secondly, I will present tailor-made computational methods for training large-scale mechanistic models of cellular pathways using multi-omic data. This includes the application of mini-batch optimisation algorithms to scalable mechanistic modeling. Thirdly, I will outline methods for the use of low-grade data, e.g. information about qualitative properties, for the development of quantitative mechanistic models. </p></div></div></section><section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Torbjörn Nordling</span><h3></h3><span class="italic">Multistate Models to Predict Disease Progression of Type 2 Diabetes</span></h3></div></section><div class="body-text mt-4"><div><p>Drug development for therapeutic areas of progressive diseases is challenging as the natural changes in the disease over time needs to be accounted for in order to generate reasonable predictions the expected gained with a particular pharmacological intervention. Type 2 diabetes is commonly described as a progressive disease where the progression starts before the disease manifests with loss of glycemic control. In this presentation, I will describe how we have used multistate models of sometimes competing events to develop predictive models of diabetes onset as well a progression of long-term complications. </p></div></div></section><section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Matthias König</span><h3></h3><span class="italic"></span></h3></div></section><div class="body-text mt-4"><div><p>TBA</p></div></div></section><section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Andreas Hellander</span><h3></h3><span class="italic"></span></h3></div></section><div class="body-text mt-4"><div><p>TBA</p></div></div></section></div></section></section><section class="mt-4 lg:pb-4 lg:border-b lg:border-gray-400 w-full"><div id="talks" class="h-0" style="bottom: 100px; position: relative;"></div><h2 class="section-header">Talks</h2><div class="mt-4 pb-4 border-b border-gray-400"><ul class="list-decimal"><li class="ml-8"><span class="font-semibold">Sebastian Sten</span>,
  <span class="text-1">Drug metabolism and pharmacokinetics (DMPK), Research and Early Development, Cardiovascular, Renal and Metabolism (CVRM), BioPharmaceuticals R&amp;D, AstraZeneca</span><pre></pre><a href="#talks-sebastian-sten" class="text-mbm-green hover:text-mbm-green-light">
    Case study in modern pharmaceutical drug development: how mathematical modeling can aid in unraveling systems-level insights and help guide rational drug design
  </a></li><li class="ml-8"><span class="font-semibold">Gustav Lindwall</span>,
  <span class="text-1">Mathematical Sciences, Chalmers Univeristy of Technology</span><pre></pre><a href="#talks-gustav-lindwall" class="text-mbm-green hover:text-mbm-green-light">
    Fast and precise inference on diffusivity in particle systems
  </a></li><li class="ml-8"><span class="font-semibold">Sonja Mathias</span>,
  <span class="text-1">Department of Information Technology, Uppsala Universitet</span><pre></pre><a href="#talks-sonja-mathias" class="text-mbm-green hover:text-mbm-green-light">
    Contributions of cell behavior to column growth in embryonic cartilage
  </a></li><li class="ml-8"><span class="font-semibold">Oscar Arrestam</span>,
  <span class="text-1">Department of Biomedical Engineering, Linköping University</span><pre></pre><a href="#talks-oscar-arrestam" class="text-mbm-green hover:text-mbm-green-light">
    Digital twin predicting diet response before and after long-term fasting
  </a></li><li class="ml-8"><span class="font-semibold">Lucas Hedström</span>,
  <span class="text-1">Department of Physics, Umeå University</span><pre></pre><a href="#talks-lucas-hedström" class="text-mbm-green hover:text-mbm-green-light">
    Modelling chromosome-wide target search
  </a></li></ul></div><div class="lg:grid lg:grid-flow-row lg:grid-cols-2 lg:gap-x-24"><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div id="talks-sebastian-sten" class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Sebastian Sten</span><h3></h3><span class="text"></span><span class="italic">Case study in modern pharmaceutical drug development: how mathematical modeling can aid in unraveling systems-level insights and help guide rational drug design</span></h3></div><div class="body-text speaker-description mb-8"><div><p>Modern drug development typically employs mathematical modeling to generate quantitative relationships between drug exposure (pharmacokinetics), the interaction between the drug and its intended target (target engagement), and finally, how this drug-target interaction affects downstream functional readouts (pharmacodynamics). These models range from simple linear regression to complex multi-variable differential equation models, with potentially many unknown or unmeasurable parameters that needs to be inferred from in vitro and in vivo data. When successful, these quantitative relationships (models) are used to design experimental studies, unravel insights regarding the biological system, and ultimately guide compound and efficacious dose selection. Here in, we present a case study of how these concepts were applied to agonists towards a kidney-enriched receptor. Suggested agonists exhibited dose-dependent elimination, exhibiting less clearance at high drug concentrations. By applying mechanistic modeling based on in vitro/in vivo data, the root-cause of the saturation was predicted to be due to fast receptor recycling. The model predictions were used to successfully design and predict new in vivo experiments, confirming the presence of receptor-driven drug clearance. Furthermore, we show that a common rationale – improved in vitro affinity translates to improved in vivo potency – is not sufficient in this case. In contrast, in-line with the model predictions, the actual in vitro drug-receptor binding kinetics (association/disassociation rate) determine the in vivo potency of these agonists. In summary, mathematical modeling in drug development plays an important role in aiding a correct translation between in vitro drug properties and in vivo pharmacokinetics/pharmacodynamics, and helps identify desirable drug properties.</p></div></div></section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div id="talks-gustav-lindwall" class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Gustav Lindwall</span><h3></h3><span class="text"></span><span class="italic">Fast and precise inference on diffusivity in particle systems</span></h3></div><div class="body-text speaker-description mb-8"><div><p>Particle systems made up of interacting agents is a popular model used in a vast array of applications, not the least in biology where the agents can represent everything from single cells to animals in a herd. Usually, the particles are assumed to undergo some type of random dynamics, and a popular way to model this is by using Brownian motion. The magnitude of random motion is often quantified using mean squared displacement, which provides a simple estimate of the diffusion coefficient. However, this method often fails when data is sparse or interactions between agents frequent.</p></div><div><p>In order to adress this, we derive a conjugate relationship in the diffusion term for large interacting particle systems undergoing isotropic diffusion, giving us an efficient inference method. The method accurately accounts for emerging effects such as anomalous diffusion stemming from mechanical interactions. We apply our method to a complex model with a large number of interacting particles, and the results are contrasted with a naive mean square displacement-based approach. We find a significant improvement in performance when using the higher-order method over the naive approach. This method can be applied to any system where agents undergo Brownian motion and will lead to improved estimates of diffusion coefficients compared to existing methods.</p></div></div></section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div id="talks-sonja-mathias" class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Sonja Mathias</span><h3></h3><span class="text"></span><span class="italic">Contributions of cell behavior to column growth in embryonic cartilage</span></h3></div><div class="body-text speaker-description mb-8"><div><p>During early development, cartilage provides shape and stability to the embryo while serving as a blueprint for the skeleton. Correct formation of embryonic cartilage is hence essential for healthy  development. In vertebrate cranial cartilage, it has been shown that a flat and laterally extended sheet-like macroscopic geometry is linked to a regular microscopic geometry consisting of tightly packed, short, transversal clonar columns. Which mechanical  interactions and cell behaviours contribute to the generation and  maintenance of this geometry during embryogenesis remains unclear. In this talk, we apply a three-dimensional cell-based computational model to investigate mechanical principles contributing to column  formation. Our model accounts for clonal expansion, anisotropic proliferation and the geometrical arrangement of progenitor cells in space. We confirm that oriented cell divisions and repulsive mechanical interactions between cells are key drivers of column  formation. In addition, column formation benefits from increased extracellular matrix in the initial configuration and column maintenance is facilitated by sequential proliferative phases. Our model thus correctly predicts the dependence of local order on division orientation and tissue thickness. Our study presents the first cell-based simulations of cell mechanics during cranial cartilage formation and will be useful in future studies on the  formation and growth of other cartilage geometries.</p></div></div></section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div id="talks-oscar-arrestam" class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Oscar Arrestam</span><h3></h3><span class="text"></span><span class="italic">Digital twin predicting diet response before and after long-term fasting</span></h3></div><div class="body-text speaker-description mb-8"><div><p>Today, there is great interest in diets proposing new combinations of macronutrient compositions and fasting schedules. Unfortunately, there is little consensus regarding the impact of these different diets, since available studies measure different sets of variables in different populations, thus only providing partial, non-connected insights. We lack an approach for integrating all such partial insights into a useful and interconnected big picture. Herein, we present such an integrating tool. The tool uses a novel mathematical model that describes mechanisms regulating diet-response and fasting metabolic fluxes, both for organ-organ crosstalk, and inside the liver. The tool can mechanistically explain and integrate data from several clinical studies, and correctly predict new independent data, including data from a new study. Using this model, we can predict non-measured variables, e.g. hepatic glycogen and gluconeogenesis, in response to fasting and different diets. Furthermore, we exemplify how such metabolic responses can be successfully adapted to a specific individual's sex, weight, height, as well as to the individual's historical data of metabolite dynamics. This thus enables an offline digital twin technology. </p></div></div></section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div id="talks-lucas-hedström" class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Lucas Hedström</span><h3></h3><span class="text"></span><span class="italic">Modelling chromosome-wide target search</span></h3></div><div class="body-text speaker-description mb-8"><div><p>Experiments suggest that proteins find DNA target sequences by diffusing along with the DNA, occasionally unbinding and rebinding to other further away sites. This process is known as facilitated diffusion, which has been described by multiple biophysical models. In those models, they treat the basepair sequence as an energy landscape, with the protein acting as a diffusing particle looking for one of the landscape's valleys.</p></div><div><p>Because of the vast size of DNA, these simulations cannot be generalized to genome-wide searches. Instead, the models that deal with large-scale search omit sequence motifs and envision chromosomes as a 3D contact network that guides the protein across the chromosome. While applicable, these models cannot predict binding rates to specific motifs. Therefore, to model the chromosome-wide search of "real proteins", we need to connect the small- and large-scale.</p></div><div><p>We present a model that connects search at a basepair- and chromosome-wide level to simulate search times for specific proteins looking for target motifs. At the basepair level, we use an established two-state facilitated diffusion model, and at the large scale, we build on previous work modelling the protein search as a random walk on a weighted network representing the 3D interaction of the chromosome network from Hi-C experimental data.</p></div><div><p>Our method allows us to calculate the search rates for proteins in chromosome-wide search, considering factors such as the variance of the energy landscape and target placements. Preliminary results show that the search time is dependent not only on the fine-scale basepair dynamics but also on the inherent placement of target sequences on the chromosome's 3-dimensional conformation.</p></div></div></section></div></section><section class="mt-4 lg:pb-4 lg:border-b lg:border-gray-400 w-full"><div id="posters" class="h-0" style="bottom: 100px; position: relative;"></div><h2 class="section-header">Posters</h2><div class="mt-4 pb-4 border-b border-gray-400"><ul class="list-decimal"><li class="ml-8"><span class="font-semibold">Oscar Arrestam</span>,
  <span class="text-1">Department of Biomedical Engineering, Linköping University</span><pre></pre><a href="#posters-oscar-arrestam" class="text-mbm-green hover:text-mbm-green-light">
    Digital twin predicting diet response before and after long-term fasting
  </a></li><li class="ml-8"><span class="font-semibold">Christian Simonsson</span>,
  <span class="text-1">Department of Biomedical Engineering, Linköping University</span><pre></pre><a href="#posters-christian-simonsson" class="text-mbm-green hover:text-mbm-green-light">
    A multi-scale in silico mouse model for diet induced insulin resistance
  </a></li><li class="ml-8"><span class="font-semibold">Benjamin Andersson</span>,
  <span class="text-1">Department of Biomedical Engineering, Linköping University</span><pre></pre><a href="#posters-benjamin-andersson" class="text-mbm-green hover:text-mbm-green-light">
    A new method for a priori practical identifiability
  </a></li><li class="ml-8"><span class="font-semibold">William Lövfors</span>,
  <span class="text-1">Department of Biomedical Engineering, Linköping University</span><pre></pre><a href="#posters-william-lövfors" class="text-mbm-green hover:text-mbm-green-light">
    A comprehensive mechanistic model of adipocyte signaling with layers of confidence
  </a></li><li class="ml-8"><span class="font-semibold">Moa Lundkvist</span>,
  <span class="text-1">Molecular Biology, Umeå University</span><pre></pre><a href="#posters-moa-lundkvist" class="text-mbm-green hover:text-mbm-green-light">
    Forecasting histone methylation by Polycomb complexes with minute-scale precision
  </a></li></ul></div><div class="lg:grid lg:grid-flow-row lg:grid-cols-2 lg:gap-x-24"><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div id="posters-oscar-arrestam" class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Oscar Arrestam</span><h3></h3><span class="text"></span><span class="italic">Digital twin predicting diet response before and after long-term fasting</span></h3></div><div class="body-text speaker-description mb-8"><div><p>Today, there is great interest in diets proposing new combinations of macronutrient compositions and fasting schedules. Unfortunately, there is little consensus regarding the impact of these different diets, since available studies measure different sets of variables in different populations, thus only providing partial, non-connected insights. We lack an approach for integrating all such partial insights into a useful and interconnected big picture. Herein, we present such an integrating tool. The tool uses a novel mathematical model that describes mechanisms regulating diet-response and fasting metabolic fluxes, both for organ-organ crosstalk, and inside the liver. The tool can mechanistically explain and integrate data from several clinical studies, and correctly predict new independent data, including data from a new study. Using this model, we can predict non-measured variables, e.g. hepatic glycogen and gluconeogenesis, in response to fasting and different diets. Furthermore, we exemplify how such metabolic responses can be successfully adapted to a specific individual's sex, weight, height, as well as to the individual's historical data of metabolite dynamics. This thus enables an offline digital twin technology. </p></div></div></section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div id="posters-christian-simonsson" class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Christian Simonsson</span><h3></h3><span class="text"></span><span class="italic">A multi-scale in silico mouse model for diet induced insulin resistance</span></h3></div><div class="body-text speaker-description mb-8"><div><p>Insulin resistance (IR) causes compensatory insulin production, which in humans eventually progresses to beta-cell failure and type 2 diabetes (T2D). This disease progression involves multi-scale processes, ranging from intracellular signaling to organ-organ and whole-body level regulations, on timescales from minutes to years. T2D progression is commonly studied using overfed and genetically modified rodents. However, rodents do not exhibit human T2D progression, with IR-driven beta-cell failure, and available multi-scale data is too complex to fully comprehend using traditional analysis. To help resolve these issues, we here present an in silico mouse model. This is the first mathematical model that simultaneously explains multi-scale mouse IR data on all three levels – cells, organs, body – ranging from minutes to months. The model correctly predicts new independent multi-scale validation data and provides insights into non-measured processes. Finally, we present a humanoid in silico mouse exhibiting disease progression from IR to IR-driven beta-cell failure. </p></div></div></section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div id="posters-benjamin-andersson" class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Benjamin Andersson</span><h3></h3><span class="text"></span><span class="italic">A new method for a priori practical identifiability</span></h3></div><div class="body-text speaker-description mb-8"><div><p>Mathematical modeling uses differential equations to study biological processes.  One often does not know the numerical values of the parameters before measurements are taken and these measurements are used to estimate the parameter values.  However, a parameter may be impossible to estimate this way due to the nature of the equations (structurally unidentifiable) or due to the level of noise in the signal (practically unidentifiable).  Much progress has been made in recent decades on determining structural identifiability, but the problem of practical identifiability remains a challenge.  However, it is often the case that a parameter is structurally identifiable but is not practically identifiable for any realistic level of noise.  This could be because the structural identifiability relies on high order derivatives of the output signal that are hard to accurately estimate.</p></div><div><p>We present a new algorithm that can help determine whether a parameter is practically identifiable a priori, that is, before measurements are made, by determining whether it can be estimated from derivatives of the outputs specified by the experimenter.  Additionally, we show this method can determine if a user-specified combination of states and parameters is practically identifiable.  We present a fast version of this algorithm using random integers that succeeds with high probability, and provide rigorous justification not shown in similar algorithms in the literature.</p></div></div></section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div id="posters-william-lövfors" class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">William Lövfors</span><h3></h3><span class="text"></span><span class="italic">A comprehensive mechanistic model of adipocyte signaling with layers of confidence</span></h3></div><div class="body-text speaker-description mb-8"><div><p>Adipocyte cellular signaling, normally and in type 2 diabetes, is far from fully studied. We have earlier developed detailed dynamic mathematical models for some well-studied, and partially overlapping, signaling pathways in adipocytes. Still, these models only cover a fraction of the total cellular response. For a broader coverage of the response, large-scale phosphoproteomic data is key. There exists such data for the insulin response of adipocytes, as well as prior knowledge on possible protein-protein interactions associated with a confidence level. However, methods to combine detailed dynamic models with large-scale data, using information about the confidence of included interactions, are lacking. In our new method, we first establish a core model by connecting our partially overlapping models of adipocyte cellular signaling with focus on: 1) lipolysis and fatty acid release, 2) glucose uptake, and 3) the release of adiponectin. We use the phosphoproteome data and prior knowledge to identify phosphosites adjacent to the core model, and then try to add the adjacent phosphosites to the model. The additions of the adjacent phosphosites is tested in a parallel, pairwise approach with low computation time. We then iteratively collect the accepted additions into a layer and use the newly added layer to find new adjacent phosphosites. We find that the first 15 layers (60 added phosphosites) with the highest confidence can correctly predict independent inhibitor-data (70-90% correct), and that this ability decrease when we add layers of decreasing confidence. In total, 60 layers (3926 phosphosites) can be added to the model and still keep predictive ability. Finally, we use the comprehensive adipocyte model to simulate systems-wide alterations in adipocytes in type 2 diabetes. This new method provide a tool to create large models that keeps track of varying confidence.</p></div></div></section><section class="mt-4 lg:mt-12 pb-4 lg:pb-0 border-b lg:border-b-0 border-gray-400"><div id="posters-moa-lundkvist" class="h-0" style="bottom: 100px; position: relative;"></div><div class="text-lg md:text-xl mb-4"><h3><span class="font-semibold">Moa Lundkvist</span><h3></h3><span class="text"></span><span class="italic">Forecasting histone methylation by Polycomb complexes with minute-scale precision</span></h3></div><div class="body-text speaker-description mb-8"><div><p>Polycomb proteins are critical epigenetic regulators of metazoan development. They form two major kinds of complexes, Polycomb Repressive Complex 1 (PRC1) and Polycomb Repressive Complex 2 (PRC2). PRC2 can add up to three methyl groups to Lysine 27 of histone H3 (H3K27), which is necessary for its repressive function. PRC2 functions in two ways: hit-&amp;-run, a diffusion based random methylation of the inactive genome, and by targeting to specific genes. When PRC2 targets  genes it binds to genomic elements called Polycomb Response Elements (PREs).</p></div></div></section></div></section><section class="mt-4 pb-4 border-b border-gray-400"><div id="registration" class="h-0 relative" style="bottom: 100px;"></div><h2 class="section-header">Registration</h2><p class="mt-2">
    You can participate at MBM 2022 by
  </p><ul class="mt-2 px-8 list-disc"><li>giving a plenary talk,</li><li>presenting a poster,</li><li>or simply as an observer.</li></ul><p class="mt-2">
    In the first two cases you will have to submit an <span class="font-semibold">abstract
    of max. 2000 characters</span>. Please note that the number of
    plenary speakers is limited. If you sign up for a talk but do not
    get selected for a plenary talk, you are welcome to present a poster 
    during the poster session.
  </p><p class="mt-2">
    As part of the goals for the workshop we will give preference to
    speakers on PhD and PostDoc level.
  </p><p class="mt-2">
    Please note that you can edit your registration, which allows you
    to change your participation status.
  </p><p class="mt-2">
    The <span class="font-semibold">registration deadline</span> is
  </p><ul class="mt-2 px-8 list-disc"><li>for talks and posters on May 6th, and</li><li>for observers on May 16th .</li><br></ul><p class="mt-2">
    Participation will require a fee* of 100 SEK. <br>
    Participation at the dinner will require an additional fee of 100 SEK, payment information can be found below: <br><br>

    From within Sweden:<br>
    Bank giro: 5050-0016<br>
    State &quot;17016, project 995631, MBM&quot; as reference <br><br>

    From abroad:<br>
    Danske Bank, P.O. Box 7523, SE-103 92 Stockholm, Sweden<br>
    Account number 1281 01 17 713<br>
    BIC/SWIFT: DABASESX<br>
    IBAN: SE13 1200 0000 0128 1011 7713<br>
    State &quot;17016, project 995631, MBM&quot; as reference<br><br>

    *This do not apply for MSc or undergraduate students.
  </p><form action="https://docs.google.com/forms/d/e/1FAIpQLSeFFQ27e4a0HllQHnyaMfd8Av4UFUmFKfeU0GPA0MdWEN6Cnw/viewform?usp=sf_link" target="_blank" class="mt-4 flex flex-row justify-center items-center"><input type="submit" value="Register here" class="bg-mbm-green text-gray-100 font-semibold py-2 px-6 rounded
                  hover:bg-mbm-green-dark hover:text-white cursor-pointer shadow-md"></form></section><section class="mt-4 pb-4 border-b border-gray-400 w-full"><div id="sponsor" class="h-0 relative" style="bottom: 100px;"></div><h2 class="section-header">Sponsor</h2><p>
    This event is sponsored and made possible by AstraZeneca. <br></p><img src="/assets/img/AZ_CMYK_H_COL.d0fec173.jpg" width="30%" style="margin-top:1rem"></section><section class="mt-4 pb-12"><div id="contact" class="h-0" style="bottom: 100px; position: relative;"></div><h2 class="section-header">Contact</h2><p class="mt-2"><span class="font-semibold">Organising team:</span>
  Sebastian Persson, Henrik Podéus, Nicolas Sundqvist, Kajsa Tunedal, </p><p><span class="font-semibold">Contact:</span> sebastian.persson@chalmers.se, henrik.podeus@liu.se, nicolas.sundqvist@liu.se, kajsa.tunedal@liu.se </p></section></div></div>
    <script>window.__INITIAL_STATE__={"data":null,"context":{}};(function(){var s;(s=document.currentScript||document.scripts[document.scripts.length-1]).parentNode.removeChild(s);}());</script><script src="/assets/js/app.cd4409d6.js" defer></script><script src="/assets/js/page--src-pages-index-vue.37ec974d.js" defer></script>
  </body>
</html>
