<div class="row pt-3">
  <div class="col">
    <h1 id="speakers">Invited Speakers</h1>
  </div>
</div>
<div class="row py-1">
  <div class="col-md-8">
    <h2>Matteo Barberis</h2>
    <p class="text-muted">
      Reader/Professor in Systems Biology at the University of Surrey in Guildford, UK
    </p>
    <p style="line-height: 1.1em;">
      <small>
        Matteo has recently become professor at the University of Surrey and has a background in industrial biochemistry, molecular biology and computational modelling. His research focuses on integrating experimentation and computation to gain insight on cellular organization. His main areas of focus are temporal control of gene regulation of cell cycle control and multi-scale frameworks that integrate cellular regulation.
      </small>
    </p>
  </div>
  <div class="col-md-4" style="display: flex; flex-direction: column; justify-content: center;">
    <div class="row">
      <div class="col text-center" style="height: 200px;">
        <img src="https://www.uva.nl/binaries/content/documents/personalpages/b/a/m.barberis/profilepicture/profilepicture/myprofile%3Aimage?1452700579972" alt="Matteo Barberis" class="img-fluid" style="border-radius: 10%; height: 100%;">
      </div>
    </div>
    <div class="row">
      <div class="col text-center">
        <p class="align-middle">
          Photo via
          <a href="https://www.uva.nl/en/profile/b/a/m.barberis/m.barberis.html" target="_blank" class="badge badge-primary mb-3">Personal Website</a>
        </p>
      </div>
    </div>
  </div>
  <div class="col">
    <table class="table">
      <tbody>
        <tr>
          <th scope="row" style="border: none;">Title</th>
          <td style="border: none;">Systems biology to decipher design principles of cellular networks</td>
        </tr>
        <tr>
          <th scope="row" style="border: none;">Abstract</th>
          <td style="border: none;">Continuity of life is sustained by time-dependent responses of biological networks that are associated to oscillatory behavior of their components, at a proper frequency. The eukaryotic cell cycle is such an example, with waves of enzymatic activities being tightly regulated to guarantee its frequency. Therefore, molecular designs that exhibit a proper frequency of enzymatic oscillations are inherently crucial for a timely cell cycle progression. In this talk, I will show how computational frameworks, such as kinetic modelling, Boolean modelling, and multi-scale modelling, may serve the purpose of investigating how a timely cell cycle progression is achieved, possibly resulting in the identification of novel principle of design that ensures a timely cell cycling.</td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<hr>
<div class="row pt-1 pb-3">
  <div class="col-md-8">
    <h2>Adil Mardinoglu</h2>
    <p class="text-muted">
      Professor of Systems Biology at King’s College London, UK and Royal Institute of Technology (KTH), Sweden
    </p>
    <p style="line-height: 1.1em;">
      <small>
        Dr. Adil Mardinoglu works as a Professor of Systems Biology at King’s College London, UK and Royal Institute of Technology (KTH), Sweden. His main area of expertise is in Systems Biology/Medicine, Computational Systems Biology and Bioinformatics (<a href="http://sysmedicine.com" target="_blank">http://sysmedicine.com</a>). His research focuses on the generation of the context specific biological networks for human cells/tissues as well as certain types of cancer including hepatocellular carcinoma by integrating multi-omics data. Such comprehensive networks are employed to identify novel biomarkers and drug targets that can be used for the development of effective treatment strategies for NASH/NAFLD and other obesity-associated disorders. Dr. Mardinoglu is co-founder of two companies focusing on the development of therapies for liver associated diseases.
      </small>
    </p>
  </div>
  <div class="col-md-4" style="display: flex; flex-direction: column; justify-content: center;">
    <div class="row">
      <div class="col text-center" style="height: 200px; overflow: hidden;">
        <img src="https://eas2018.com/wp-content/uploads/2017/04/Mardinoglu500.jpeg" alt="Adil Mardinoglu" class="img-fluid" style="border-radius: 10%; height: 100%;">
      </div>
    </div>
    <div class="row">
      <div class="col text-center">
        <p class="align-middle">
          Photo via
          <a href="https://eas2018.com/" target="_blank" class="badge badge-primary mb-3">86th EAS congress</a>
        </p>
      </div>
    </div>
  </div>
  <div class="col">
    <table class="table">
      <tbody>
        <tr>
          <th scope="row" style="border: none;">Title</th>
          <td style="border: none;">The use of systems biology in treatment of liver diseases</td>
        </tr>
        <tr>
          <th scope="row" style="border: none;">Abstract</th>
          <td style="border: none;">To develop novel strategies for prevention and treatment as well as to gain detailed insights about the underlying molecular mechanisms of liver diseases, it is vital to study the biological functions of liver and its interactions with other tissues and gut microbiota. Biological networks can provide a scaffold for studying biological pathways operating in the liver in connection with disease development in a systematic manner. In my presentation, I will present our recent work where biological networks have been employed to identify the reprogramming in liver physiology in response to NASH/NAFLD. I will further discuss how this mechanistic modelling approach can contribute to the discovery of biomarkers and identification of drug targets which may lead to design of targeted and effective treatment strategies.</td>
        </tr>
        <tr>
          <th scope="row" style="border: none;">Keypoints</th>
          <td style="border: none;">
            <ul>
              <li>Omics technologies are used in detailed characterization of human liver tissue in health and disease states.</li>
              <li>Biological network models are functional tools for exploring and integration of multiomics data.</li>
              <li>Systems biology uses a holistic and integrative approach for comprehensive analysis of the biological functions in healthy and diseased states.</li>
              <li>Systems Biology approaches have been successfully employed in hepatology to identify biomarkers and drug targets.</li>
              <li>These integrative tools can be used for simulation of liver tissue functions and its crosstalk with other tissues for prediction of therapeutic and side effects.</li>
            </ul>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<hr>
<div class="row pt-1 pb-3">
  <div class="col-md-8">
    <h2>Eric Libby</h2>
    <p class="text-muted">
      Associate Professor at Umeå University and part of <a target="_blank" href="http://icelab.se/">icelab</a>.
    </p>
    <p style="line-height: 1.1em;">
      <small>
        Eric's research explores the conditions that make it possible for
        multicellularity to evolve and the impact being multicellular has on
        the evolution of complex life. In essence, he is interested in how
        simple organisms evolve into complex ones. Eric's journey to IceLab
        was long and wandering. He graduated from Rice University in Houston,
        Texas with a degree in Computational and Applied Mathematics. He then
        drove to Montreal to do a PhD with Leon Glass at the Center for
        Nonlinear Dynamics at McGill University. There Eric modeled genetic
        networks, did some bioinformatics/systems biology, and taught
        mathematical biology to physiology students. Afterwards, Eric did a
        stint as a science journalist in Washington DC as part of a AAAS Mass
        Media Fellowship. He left North America for the South Pacific to do a
        postdoc with Paul Rainey in microbial evolution at the New Zealand
        Institute for Advanced Study. Missing snakes and uncuddly plants, his
        next position was as an Omidyar fellow at the Santa Fe
        Institute. After hiking in the arid, sunny Southwest he moved with his
        family to Umeå to see winter before it goes extinct from the world.
      </small>
    </p>
  </div>
  <div class="col-md-4" style="display: flex; flex-direction: column; justify-content: center;">
    <div class="row">
      <div class="col text-center" style="height: 200px;">
        <img src="{{ "/img/ericlibby.jpg" | absolute_url }}" alt="Eric Libby" class="img-fluid" style="border-radius: 10%; height: 100%;">
      </div>
    </div>
    <div class="row">
      <div class="col text-center">
        <p class="align-middle">
          Photo via
          <a href="http://icelab.se/about/team/eric-libby/" target="_blank" class="badge badge-primary mb-3">icelab</a>
        </p>
      </div>
    </div>
  </div>
  <div class="col">
    <table class="table">
      <tbody>
        <tr>
          <th scope="row" style="border: none;">Title</th>
          <td style="border: none;">The interaction between early multicellular forms and life cycles</td>
        </tr>
        <tr>
          <th scope="row" style="border: none;">Abstract</th>
          <td style="border: none;">The evolution of multicellular organisms is considered one of the major transitions of evolution. It paved the way for the evolution of new kinds of entities with greater complexity in terms of scale and organization. It has evolved many independent times in different lineages. The different origins have led to different evolutionary trajectories. Some have evolved into enormous organisms with intricate internal structures such as organs; while some have remained closer to their unicellular ancestors in terms of scale and organization. Indeed, some early forms of multicellularity have even evolved back into unicellular communities. In this talk, I will present some current experimental and mathematical models of multicellularity and discuss how the initial shape and structure of a multicellular organism can affect its evolutionary fate.</td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<hr>
<div class="row pt-1 pb-3">
  <div class="col-md-8">
    <h2>Francesco Gatto</h2>
    <p class="text-muted">
      Guest Researcher at Chalmers and Co-Founder + Chief Scientific Officer at <a href="https://elypta.com" target="_blank">Elypta</a>
    </p>
    <p style="line-height: 1.1em;">
      <small>
        Named Winner of MIT Technology Review's <em>Innovators Under 35</em> in 2018,
        Francesco co-founded Elypta in 2017, a molecular diagnostics start-up
        focussed to improve the survival outlook of cancer patients by developing
        systems biology-driven biomarkers.
        He invented a diagnostic and prognostic test for renal cell carcinoma based on
        an exceptionally accurate liquid biopsy, which leveraged on a novel systems
        biomarker for this common form of kidney cancer.
        This test recently showed unprecedented potential for other cancer types.
      </small>
    </p>
  </div>
  <div class="col-md-4" style="display: flex; flex-direction: column; justify-content: center;">
    <div class="row">
      <div class="col text-center" style="height: 200px;">
        <img src="https://iu35-prod.s3.amazonaws.com/images/Francesco_Gatto.width-3000.jpg" alt="Francesco Gatto" class="img-fluid" style="border-radius: 10%; height: 100%;">
      </div>
    </div>
    <div class="row">
      <div class="col text-center">
        <p class="align-middle">
          Photo via
          <a href="https://www.innovatorsunder35.com/the-list/francesco-gatto/" target="_blank" class="badge badge-primary mb-3">Innovators Under 35</a>
        </p>
      </div>
    </div>
  </div>
  <div class="col">
    <table class="table">
      <tbody>
        <tr>
          <th scope="row" style="border: none;">Title</th>
          <td style="border: none;">From systems biology research to a diagnostic start-up</td>
        </tr>
        <tr>
          <th scope="row" style="border: none;">Abstract</th>
          <td style="border: none;">An ultimate goal of fundamental research is to improve the life of our people. Systems biology research can help patients to live better and longer by integrating data on their disease into personalized treatment. This is the core of precision medicine and requires innovation in therapeutics and diagnostics. During this talk, I will show an empirical model to make precision medicine a reality, a path set forth by innovation and a path that scientists have a privileged if not an exclusive access to. I will illustrate this model within the context of my own journey with Elypta, a molecular diagnostics start-up I cofounded in 2017 starting from a systems biology research project initiated during my doctoral studies.</td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<hr>
<div class="row pt-1 pb-3">
  <div class="col-md-8">
    <h2>Peter Gennemark</h2>
    <p class="text-muted">
      Principal scientist at <a href="https://www.astrazeneca.com/our-science/gothenburg.html" target="_blank">AstraZeneca R&amp;D</a> and Adjunct Assoc Prof in Systems Biology at Linköping University
    </p>
    <p style="line-height: 1.1em;">
      <small>
        Peter works on mathematical modelling of biological systems and their interactions with drugs (systems biology and pharmacokinetics/pharmacodynamics). He is focusing on systems relevant for metabolic and cardiovascular diseases. In his work, he develops and analyses mathematical models of different granularity, and apply them in the preclinical phase of drug discovery. He collaborates closely on systems biology research topics with Gunnar Cedersund at Linköping University, for instance on modelling of micro-physiological (organ-on-a-chip) systems.
      </small>
    </p>
  </div>
  <div class="col-md-4" style="display: flex; flex-direction: column; justify-content: center;">
    <div class="row">
      <div class="col text-center" style="height: 200px;">
        <img src="https://warwick.ac.uk/fac/sci/eng/research/biomedical/impact/project_members/petergennemarkphoto.jpg" alt="Peter Gennemark" class="img-fluid" style="border-radius: 10%; height: 100%;">
      </div>
    </div>
    <div class="row">
      <div class="col text-center">
        <p class="align-middle">
          Photo via
          <a href="https://warwick.ac.uk/fac/sci/eng/research/biomedical/impact/project_members/" target="_blank" class="badge badge-primary mb-3">IMPACT - Project members</a>
        </p>
      </div>
    </div>
  </div>
  <div class="col">
    <table class="table">
      <tbody>
        <tr>
          <th scope="row" style="border: none;">Title</th>
          <td style="border: none;">Mathematical modelling in preclinical drug discovery</td>
        </tr>
        <tr>
          <th scope="row" style="border: none;">Abstract</th>
          <td style="border: none;">The talk generally describes what type of mathematical models are used in the preclinical phases of drug discovery in the pharmaceutical industry, and how these models are implemented and applied in the discovery process. Focus is on how to translate the pharmacokinetics and the pharmacodynamics of a drug candidate between in vitro systems, animal models and humans. The talk includes several specific examples from the metabolic and cardiovascular areas.</td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<hr>
<div class="row pt-1 pb-3">
  <div class="col-md-8">
    <h2>Marina Rafajlović</h2>
    <p class="text-muted">
      Assistant Professor at Gothenburg University, Marine Sciences
    </p>
    <p style="line-height: 1.1em;">
      <small>
        Marina has recently become Assistant Professor at the Department of Marine Sciences, University of Gothenburg, and she is a member of the <a href="www.cemeb.science.gu.se" target="_blank">Centre for Marine Evolutionary Biology, University of Gothenburg</a>. Marina has a background in Physics and Theoretical Biology. Her main research interests are within Evolutionary Biology. Specifically, she is interested in the mechanisms of speciation, species' range expansions and the evolutionary advantages of sexual reproduction. Marina's research method involves mathematical modelling (individual-based computer simulations) with model results serving as an input against which empirical genetic data are tested. Marina has recently been awarded a grant by the Hasselblad Foundation for developing her research on the evolution of populations undergoing range shifts, specifically focusing on short- and long-term potential for local genetic adaptation in expanding populations.
      </small>
    </p>
  </div>
  <div class="col-md-4" style="display: flex; flex-direction: column; justify-content: center;">
    <div class="row">
      <div class="col text-center" style="height: 200px;">
        <img src="https://i1.rgstatic.net/ii/profile.image/342395414695938-1458644836948_Q512/Marina_Rafajlovic.jpg" alt="Marina Rafajlovi" class="img-fluid" style="border-radius: 10%; height: 100%;">
      </div>
    </div>
    <div class="row">
      <div class="col text-center">
        <p class="align-middle">
          Photo via
          <a href="https://www.researchgate.net/profile/Marina_Rafajlovic" target="_blank" class="badge badge-primary mb-3">ResearchGate</a>
        </p>
      </div>
    </div>
  </div>
  <div class="col">
    <table class="table">
      <tbody>
        <tr>
          <th scope="row" style="border: none;">Title</th>
          <td style="border: none;">Unfolding evolution using mathematical models</td>
        </tr>
        <tr>
          <th scope="row" style="border: none;">Abstract</th>
          <td style="border: none;">Evolution, the change in the heritable characteristics of biological organisms over successive generations, is the fundamental process responsible for producing and maintaining biodiversity. Over time, an evolving species may face either extinction (e.g. due to the poor potential of its individuals to cope with environmental conditions they encounter), or speciation – the species' split into two (or more) reproductively isolated species (e.g. when individual populations of the species encounter contrasting environmental conditions). Distinguishing the evolutionary mechanims leading to the former or the latter outcome is challenging to date. However, because heriatable characteristics have a genetic basis, patterns of genetic variation within a population are necessarily shaped by the evolutionary history of the population. Therefore, much of the evolutionary mechanism at work in a given population can be learned from empirical patterns of genetic variation in the population. Importantly, to interpret genetic data, it is necessary to produce theoretical expectations for how different evolutionary mechanims and their interplay shape genetic variation during evolution. In this talk, I will highlight some of the recent theoretical advances in the field, specifically focusing on new theoretical insights and open questions in speciation research.</td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<hr>
<div class="row py-3">
  <div class="col">
    <h1>Other speakers</h1>
    <div class="row py-3">
      <div class="col">
        <h3>Gunnar Cedersund</h3>
        <p class="text-muted">
          Linköping University, Department of Biomedical Engineering
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">I will in this presentation give an overview of a new technology we have developed for systems medicine, which centers around the concept of a "digital twin". A digital twin is a computer model of of a patient, which has a similar physiology as the patient. This twin can be used for both diagnosis, based on an examination of the patient as he/she is right now, and for prognosis, by doing simulations of how the patient is likely to respond to future treatments, such as medications, exercise, or a change in diet. In all instances, the user can ask the question "why" a certain prediction is obtained, since the model is based on the actual physiological and biochemical processes. Each part of the model has been carefully tested, by an iterative process of data analysis and new experiments, before it has been added to the big picture. The resulting model focuses on the interplay between processes related to metabolism, inflammation, neuronal activity, blood flow, and other processes related to the metabolic syndrome. However, thanks to the physiological structure of the model, it can readily be expanded to include additional processes, if future usages so require. In that sense, this technology goes beyond traditional narrow AI, and lays the foundation for a general AI in healthcare. Apart from healthcare applications, I will also demonstrate how our model is being used in basic research and in drug development.</td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Christoph Börlin</h3>
        <p class="text-muted">
          Chalmers University of Technology
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Title</th>
              <td style="border: none;">
                Predicting gene expression levels from transcription factor binding events through machine learning
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Binding of transcription factors (TFs) to gene promoter regions is one of the main drivers of gene regulation, but the exact mechanism and effects are not well understood. As regulation of gene expression is one of the major determinants of cellular phenotype a deeper knowledge of gene regulation is important for understanding cellular behavior and improved metabolic engineering strategies. Many TFs are only active in certain conditions and some TF binding events show no apparent effect on gene expression, making the effects difficult to decipher and to understand. Using the recently developed ChIP-exo method one can identify the binding sites and strengths for a TF with a higher resolution and with less background than previously possible.

                Machine learning approaches have been used in many applications to decipher patterns in large datasets, like speech recognition or predicting cancer treatment outcomes. The large datasets generated by ChIP-exo could also provide a strong basis for a machine learning model linking TF binding events to RNA expression levels. The successful construction of an expression-predictor should then provide valuable insight into the complex interplay of different TFs binding to the same promoter. In our project we collected ChIP-exo data from Saccharomyces cerevisiae for a number of different TFs in four different metabolic conditions. Using tree based machine learning methods we are generating a gene expression predictor that can then be used to predict expression levels for altered promoters in metabolic engineering projects.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Sandro Stucki</h3>
        <p class="text-muted">
          Gothenburg University, Department of Computer Science and Engineering (CSE)
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Title</th>
              <td style="border: none;">
                Rule-Based Modelling of Bio-Chemical Networks
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Rule-based modeling (RBM) is a paradigm for describing combinatorially complex stochastic processes with an underlying dynamic network structure through a finite set of rewrite rules. In this talk, I will give an overview of RBM – illustrating how it can be used to model biochemical processes in particular – and present some preliminary work on the automated generation of rate-equations and ODEs tracking higher-order statistics from a large class of rule-based systems.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Anass Bouchnita</h3>
        <p class="text-muted">
          Uppsala University
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Title</th>
              <td style="border: none;">
                Overcoming drug resistance in multiple myeloma: insights from a hybrid discrete-continuous model
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Authors</th>
              <td style="border: none;">
                Anass Bouchnita, Vitaly Volpert, Mark J. Koury, Andreas Hellander
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Drug resistance is one of the main challenges that limit the therapeutic potential of existing cancer treatments. Therefore, designing therapeutic strategies that restore the sensitivity of cancer cells would be greatly beneficial. In this talk, we present a multiscale model devoted to the interaction between multiple myeloma (MM) and potential treatments. In this model, MM cells are represented as individual objects that move, divide, and die by apoptosis. Their fate is determined by intracellular and extracellular regulation, as well as the treatment. We use the model to explore the effects of treatment protocols consisting of a tyrosine-kinase inhibitor (TKI) and a pentose phosphate pathway (PPP) inhibitor. Numerical simulations suggest an interval for the daily dosing of the PPP inhibitor that maximizes the anti-myeloma effects of the TKI. Furthemore, they demonstrate that the combination of high-dose pulsatile TKI treatment with high dose continuous PPP inhibitor therapy can potentially eradicate the tumor. The model predictions can be used to design clinical trials in the future.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Niek Welkenhuysen</h3>
        <p class="text-muted">
          Chalmers University of Technology
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Crosstalk in biology is any phenomenon by which an intracellular signal transmitted in one signalling pathway creates an effect in another pathway. Typically, to study these pathway interconnections perturbation of the signalling system is involved. Unfortunately, perturbations often produce noise which causes a major challenge in identification of interconnections. Employing systems biology approach by developing appropriate models can provide minimal intervention in these systems. Typically, signalling pathways are modelled by ordinary differential equations. However, creating dynamical models of signalling pathways of a realistic size is still an obstacle. To overcome this problem, we developed a vector-based reaction-contingency Boolean logic model of the yeast Saccharomyces cerevisiae nutrient sensing pathways cAMP-PKA, Snf1, Snf3-Rgt2 and TOR pathway. To increase the information content of Boolean model from simple 'active' and 'inactive', we assigned a vector to each component describing following features: localization, phosphorylation status, guanylation status and DNA binding status. We found during the gap filling process that most lacking components are phosphatases which exposes a lack of knowledge on phosphatases involved in the sensing process. The gap filling process also identified crosstalk from the PKA and Snf1 pathway to other pathways as a vital aspect to make the model switch between nutrient conditions. We simulated the model with known crosstalk combinations. Subsequent analysis of the simulations that the crosstalk from the Snf1 pathway to the Rgt2/Snf3 pathway contributes to the robustness of this signalling network. Altogether, this work shows that network interconnections lead to the robustness of signalling pathways.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Annikka Polster</h3>
        <p class="text-muted">
          University of Örebro
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                A growing body of evidence confirms the important role gut microbiota have in health and disease, but the detailed mechanisms of interaction between gut microbiota and host are unknown. Given the difficulties in experimentally studying these interactions both in vivo and in vitro, computational approaches useful for determining which microbial species interact with which host factors may greatly enhance our understanding of this interaction and identify which mechanisms are most promising targets for detailed experimental studies. We have developed a stepwise analysis pipeline, in which rank correlations between microbial species and host factors are utilized as a proxy for estimating the grade of interaction between the respective variables, and plotted as network graphs to facilitate the intuitive interpretation of these interactions. The relevance of the respective interaction in the pathophysiology of diseases is estimated by both statistical testing and comparisons between healthy and diseased individuals, as well as by creating a multiplex network graph associating network motifs to key symptoms.

                We have tested the applicability of this pipeline in a pilot study on humans, utilizing data from healthy individuals (HC) as well as patients with Irritable Bowel Syndrome (IBS), a gastrointestinal disorder characterized by various chronic symptoms and incompletely understood pathophysiology.

                In this pilot study, network motifs, indicating close associations of the respective variables, were identified in both groups. A comparison between HC and IBS showed distinct differences in network structure, indicating differences in host-microbiota interaction between these two groups. Network motifs which were present in IBS but not in HC were additionally linked to key symptoms, supporting the relevance of these variables for understanding IBS pathophysiology. This pilot study therefore confirmed our stepwise integrative analysis pipeline as a useful approach for improving our understanding of host-microbiota interaction in health and disease.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Marco Longfils</h3>
        <p class="text-muted">
          Chalmers University of Technology and University of Gothenburg
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Since the introduction of image correlation spectroscopy (ICS) (Petersen et al. 1993), the family of fluorescence fluctuations spectroscopy methods has grown richer with the appearance of novel variants of previous techniques. All ICS methods analyse the fluctuations of the fluorescence signal in order to quantify relevant parameters of the processes that cause the fluctuations. These methods can be classified based on whether fluctuations are considered over time and in space. Raster image correlation spectroscopy (RICS) was first introduced in Digman et al. (2005) as a method to study molecular transport in solution and in cells. RICS registers both the temporal and the spatial intensity fluctuations by scanning a laser beam over a sample, and exploits the fast pixel-to-pixel and line-to-line sampling to analyze the fluctuations. The fluorescence intensity fluctuations are caused by the dynamics of the particles, including, but not limited to, diffusion, flow, and binding. In RICS, the fluorescence fluctuations  autocorrelation function is mainly used to determine macroscopic phenomenological parameters, such as the diffusion coefficient and the concentration of the molecule of interest. These parameter estimates can be obtained by fitting the model autocorrelation function to the empirical autocorrelation computed from a series of images. In my talk, I will give an introduction to the statistical analysis of fluorescence correlation spectroscopy methods and in particular RICS. In addition, I will show how these techniques can be used to measure diffusion of proteins inside cells.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Philip Dalsbecker</h3>
        <p class="text-muted">
          University of Gothenburg
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                There is an ever-growing need for safe and reliable medicines in the world. Drug development is a lengthy and costly process, largely due to the lack of reliable models to use during the development process. The liver plays a key role in metabolizing most foreign compounds such as drugs, and therefore its main parenchymal cell type – hepatocytes – are used in drug development to study efficacy and safety. However, the culture environment matters greatly, and conventional 2D cultures fall short in both viability and functionality. Here, I present my research on liver-on-a-chip, a fully perfused 3D culture model for culturing both immortalized and induced stem cell-derived hepatocytes, refined through flow simulations and experiments.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Iván Domenzain</h3>
        <p class="text-muted">
          Chalmers University of Technology
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Recently, a constraint-based method for modelling metabolism that incorporates genome-scale protein abundance measurements and kinetic parameters, as maximum capacity constraints, (GECKO) has been developed. It has been successfully applied for predicting overflow metabolism and stress responses on Saccharomyces cerevisiae cells. Although the method was conceived in a general way, its main limitations are the heterogeneity on the availability of kinetic data for different organisms and the intrinsic high variability on absolute proteomics datasets.
                Here I present an extended and generalized version of the GECKO toolbox. The algorithm is able to generate enzyme constrained models (ecModels) with high parameters coverage, even for poorly studied organisms, based on phylogenetic distance criteria. For improving the predictive power of the models, an iterative and data-based parameter curation procedure has been integrated into the model construction algorithm. The toolbox requires minimal human interaction and provides high-quality models, calibrated with experimental data and ready for simulations.
                As case studies, the pipeline has been tested on 4 different microbial organisms (3 yeast species and E. coli), yielding feasible models with high flux variability reduction due to the incorporated additional constraints. Moreover, the pipeline has been extended and adapted for the generation of ecModels for specific human cell-lines or tissues, this has allowed a detailed exploration of the metabolic network of 11 different cancer cell-lines, yielding numerical predictions that reasonably agree with experimentally measured growth rates and metabolites exchange fluxes, all of this with a minimal amount of constraints compared to classical genome-scale metabolic modeling.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Angelo Limeta</h3>
        <p class="text-muted">
          Chalmers University of Technology
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Immune checkpoint inhibitors (ICIs) have revolutionized the treatment outcomes for several cancer types that were previously considered incurable. This includes metastatic melanoma, where treatment with ICIs has shown exceptional response and possibly curative effects in a subset of patients. Nonetheless, the majority of patients undergoing ICI therapy do not respond at all. Recently, several studies have identified the patient’s gut microbiota composition as a main factor for mediating response to treatment. We are performing a meta-analysis of all published fecal metagenomic whole genome shotgun (WGS) data sets for melanoma patients undergoing ICI therapy. WGS data provides a systematic view of all the species present in the gut and allows us to determine the taxonomic, genomic, and functional differences between responders and non-responders. Identifying these differences could lead to better models for treatment response prediction, and design of pro/pre-biotic supplements for ICI therapy.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Valentina Fermanelli</h3>
        <p class="text-muted">
          Chalmers University of Technology and University of Gothenburg, Department of Mathematical Sciences
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Authors</th>
              <td style="border: none;">
                Valentina Fermanelli, Martin Adiels
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Lipoproteins are a family of lipid carrying particles in the blood which contains the vast amount of plasma triglycerides and cholesterol. Apolipoproteins are proteins attached to the surface of these lipoproteins and confer them different properties.
                We study how fast three apolipoproteins (ApoE, ApoC3 and ApoA4) are produced and eliminated, by analyzing experimental data.
                In the experiment, a stable isotope of leucine (labelled molecule) is given to a group of volunteers and the ratio of labelled leucine against total leucine is measured over 24 hours in two different stages, free leucine and leucine in the three apolipoproteins.
                We use nonlinear mixed effects (NLME) models to perform the estimations.
                Our aims of the study include: How does the NLME perform when comparing separate and combined models of apolipoprotein kinetics? How are the kinetics of the apolipoproteins correlated to each other and to other characteristics of the individuals?
                The results show that the correlation structure of the parameters is not strongly affected by the modelling choice.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Elin Nyman</h3>
        <p class="text-muted">
          Linköping University
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Title</th>
              <td style="border: none;">
                Perturbation biology links temporal protein changes to drug responses in a melanoma cell line
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Many melanoma tumors initially respond to targeted drugs. Such targeted drugs are developed to attack growth signaling pathways which are increased from mutations in the tumor cells. However, the response to targeted drugs in melanoma is often temporary due to short-term adaptation and long-term resistance development. To get closer to understand these mechanisms, we study in detail the short term adaptation of cancer cells to a set of targeted drugs alone and in combination. Specifically, we systematically measure the time-resolved response of 124 key proteins to 54 targeted drug combinations over 72 hours in the A2058 melanoma cell line, which carries a common mutation (V600E) in the gene BRAF. Furthermore, the cell line is resistant to targeted drugs that are used in the clinic, and therefore a good model to study drug resistance. We also measure cell growth and apoptosis as readouts for drug efficacy. This rich and unique dataset enables us to build ordinary differential equations based network models that capture not only the temporal patterns of the measured proteins, but also allows to predict the effect of unseen drug combinations. Based on the network models, we propose novel perturbations that are predicted to overcome the short-term adaption by targeting combinations of the proteins STAT5a, EGFR, and IRS1. The insights from our data-derived modeling efforts contributes to the identification of drug targets that may overcome drug resistance in melanoma patients.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
</div>
<hr>
<div class="row py-3">
  <div class="col">
    <h1>Poster presenters</h1>
    <div class="row py-3">
      <div class="col">
        <h3>Sonja Mathias</h3>
        <p class="text-muted">
          Uppsala University
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Title</th>
              <td style="border: none;">
                Studying the formation of cartilage sheets using cellular mechanics simulations
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Authors</th>
              <td style="border: none;">
                Sonja Mathias, Andreas Hellander, Igor Adameyko
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                During embryonic development cartilaginous structures develop that later densify into bone and thus form the basis for the embryo's skeleton. Understanding the cellular dynamics responsible for the correct shaping and growth of the cartilage is of high importance. We build a computational model of a cartilage sheet to study the interplay between different mechanisms. It combines a center-based model for the cellular mechanics with rules governing the cellular behavior based on biological observations. Our preliminary results confirm that oriented cell division is necessary for obtaining the column-like structures observed in flat cartilage sheets in vivo. Further questions to study include how the smooth border of the sheet is controlled, and how different cartilage shapes depend on the initial placement of the progenitor cells.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Nicolas Sundqvist</h3>
        <p class="text-muted">
          Linköping University, Department of Biomedical Engineering
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                One of the key parameters in understanding complex metabolic networks are the metabolic fluxes, which are the rates of conversion of metabolic intermediates. Currently, one of the main approaches for determining these fluxes is metabolic flux analysis (MFA), in which isotope-labelled compounds are introduced into the system and measured. Mathematical models are then used to calculate a prediction of the systems flux configuration, based on the distribution of these isotopic labels. However, the current paradigm of MFA lack established methods for validating that a model can accurately predict quantities for which there are no experimental data. In this study we propose and evaluate a new approach for validating the predictions of metabolic fluxes from 13C MFA models and showing that such predictions can be well determined. A mathematical model that covers parts of the central human metabolism was created and evaluated with regards to the model’s ability to predict a validation dataset, having been fitted to an estimation dataset. Furthermore, if the model could predict metabolites for which it had not been fitted towards, the uncertainties of such predictions were determined. The results conclusively showed that 13C MFA models can be validated against experimental data that the model has never seen before, with well determined prediction uncertainties. This allows for greater trust to be placed in the models and could allow them to take on a larger role in the development process of new treatments or medicine for various diseases.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>David Bergenholm</h3>
        <p class="text-muted">
          Chalmers University of Technology
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                In response to environmental changes the cellular functions are reprogrammed. Transcription factors play an essential role in this regulation. Through physical contact with either proteins or DNA the transcription factor alters the gene expression. Reconstruction of transcriptional regulatory networks require in depth understanding of these protein-protein or protein-DNA interactions. We performed whole genome analysis of the protein-DNA binding at high resolution (Chip-exo) of twenty-one Saccharomyces cerevisiae transcription factors, in response to four different environmental conditions in chemostat cultures. Because the targets of a transcription factor may differ depending on environmental conditions, the four different conditions will enable us to find the responses of each transcription factor. Combining the binding profiles and targets of each transcription factor with transcriptome data and metabolic pathways we can start to unveil the complexity of the regulatory network machinery.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Najmeh Abiri</h3>
        <p class="text-muted">
          Lund University
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Dealing with missing data in any data analysis is inevitable. Although there are powerful imputation methods that address this problem, there is still much room for improvement. Recently, Variational auto-encoders (VAEs) received a lot of attention as imputation methods. VAEs are generative models that use variational inference combined with auto-encoders to learn the underlying representation of data by reproducing them. This property makes it possible to use them as imputation methods.
                In this study, we propose a new structure for the variational autoencoder (VAE) prior, with the weakly informative multivariate Student-t distribution. In the proposed model all distribution parameters are trained, thereby allowing for a more robust approximation of the underlying data distribution. We used Fashion-MNIST data in two experiments to compare the proposed VAE with the standard Gaussian prior. Both analyses showed a better reconstruction of the images with VAE using Student-t prior distribution. Possible applications of the method are in proteomics (e.g. protein profiling of breast cancer cell lines) and gene expression measurement for somatic mutations in lung cancer.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Jacob Leander</h3>
        <p class="text-muted">
          Chalmers University of Technology and Fraunhofer Chalmers Centre
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Title</th>
              <td style="border: none;">
                A parameter estimation framework for nonlinear mixed effects models with ordinary and stochastic differential equations implemented in Mathematica
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Authors</th>
              <td style="border: none;">
                Jacob Leander, Joachim Almquist, Helga Kristín Ólafsdóttir, Anna Johnning, Mats Jirstrand
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                In many applications within biology and medicine, measurements are gathered from several entities in the same experiment (e.g. patients, cells, tissue samples) with the aim of characterizing for example response to a specific drug treatment or other stimuli. To characterize the variability in response between entities, the nonlinear mixed effects model is a suitable statistical model. A nonlinear mixed effects model enables quantification of both within- and between subject variability.

                Parameter estimation in a nonlinear mixed effects model is not straightforward, due to the intractable expression of the likelihood function. Approximations of the likelihood function or other MCMC-based approaches are typically used to infer parameters from data. Several parameter estimation software are available, but few have reached the academic community.

                In this work, we present a framework for parameter estimation in nonlinear mixed effects models with a longitudinal response model described by ordinary differential equations. The framework is implemented in the computational environment Mathematica. The estimation algorithm is based on the FOCE approximation of the likelihood, and the symbolic capabilities of Mathematica is used to enable a gradient-based optimization method using sensitivity equations (denoted S-FOCE). Moreover, extension to stochastic differential equations is also supported.

                We exemplify the framework using data from a simulated pharmacokinetic (drug concentration) model. The implementation enables a user-friendly framework for parameter estimation in nonlinear mixed effects model with ordinary or stochastic differential equations.  Moreover, the proposed method has been shown to reduce computational time and has lately been implemented in the industry-standard software NONMEM 7.4.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>William Lövfors</h3>
        <p class="text-muted">
          Linköping University
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Title</th>
              <td style="border: none;">
                The First Phosphoproteome-wide Mechanistic Model of Insulin Signaling
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Authors</th>
              <td style="border: none;">
                William Lövfors, Elin Nyman, and Gunnar Cedersund
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Over the last 5 years our group have developed a detailed mechanistic model, which can explain the core mechanisms involved in insulin signaling from insulin stimuli to glucose uptake. However, the model in its current state is only able to explain 10 proteins, with up to a maximum of 2 phosphorylations per protein.
                We are developing a new method where small-scale mechanistic models, such as the described insulin model, can be scaled to omics-sized models without losing the mechanistic properties. This scaling can be achieved by using high-throughput data in combination with established prior knowledge in order to expand the "core model" in an iterative fashion. In detail, the core-model is expanded in an iterative fashion by using the core-model as input and attempting to add as many proteins as possible (with constrains on how well the model must explain the data) using a list of possible interactions from prior knowledge. These newly added proteins are then used as new potential inputs for the next iteration. This process is repeated until no more proteins can be added.
                In the end we have an omics-scale mechanistic model, which is in agreement with data and prior-knowledge. This model can then e.g. be used for finding new drug targets or to study the behavior of phosphoprotein levels in high-resolution.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Christian Simonsson</h3>
        <p class="text-muted">
          Linköping University, Department of Biomedical Engineering (IMT)
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Title</th>
              <td style="border: none;">
                Mathematical modeling of adiponectin release in white adipocytes in response to adrenergic stimulus
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Authors</th>
              <td style="border: none;">
                William Lövfors (1,2), Christian Simonsson (1), Elin Nyman (1), Charlotta Olofsson (3), Gunnar Cedersund (1)
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Affiliations</th>
              <td style="border: none;">
                (1) Department of Biomedical Engineering (IMT), Linköping university. (2) Department of Mathematics (MAI), Linköping university. (3) Department of Physiology/Metabolic Physiology, Institute of Neuroscience and Physiology, University of Gothenburg.
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Adiponectin is a hormone secreted by the adipose tissue, involved in the regulation of glucose levels and fatty acid breakdown. Furthermore, adiponectin has insulin sensitizing properties and an inverse relationship to adiposity, thus higher circulating levels of the hormone has been associated with a lower risk of type 2 diabetes (T2D). Since T2D is a prevalent, harmful and preventative disease, it is of importance to understand how adiponectin is secreted. Previously, we have developed a mathematical model of how adiponectin is released in response to Ca2+, cAMP and ATP during patch-clamp experiments in white adipocytes (3T3-L1 cells). We are now extending this model to also account for extracellular stimulus of epinephrine and CL 316,243 (an adrenergic β3 receptor agonist). By doing this extension we are shifting the model towards being more physiologically relevant. Thus, the secretion model has the possibility to be connected to other already established adipocyte models. Also, since some of these adipocyte models are able to simulate T2D, we could possibly predict the change of adiponectin release in T2D, or how a change in adiponectin release affects e.g. glucose uptake in other tissues.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Gustav Lindwall</h3>
        <p class="text-muted">
          Mathematical Sciences, Chalmers University of Technology and University of Gothenburg
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Title</th>
              <td style="border: none;">
                An interacting position jump model of glioblastoma growth
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Authors</th>
              <td style="border: none;">
                Gustav Lindwall and Philip Gerlee
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                The brain tumour glioblastoma multiforme is characterized by diffuse and infiltrative growth into surrounding brain tissue. Efforts to model tumour growth using a mathematical framework have made great strides in the last few decades, and a wide array of mathematical methods have been applied to this emerging field. This includes phenomenological macroscopic models, data-driven statistical undertakings and physics-inspired bottom-up models utilizing stochastic processes. In this poster, we present two models mainly inspired by the theory of interacting particle systems. We let each individual cancer cell constitute a particle. We model cell division and death as Poisson processes with rates that depend on the local cell density. The cells are subject to a random isotropic motion modelled by mutually independent Wiener processes, but interact according to a pair potential which accounts for both volume exclusion and adhesion. The second model we consider incorporates phenotype switching between a resting and motile state. Here only resting cells are allowed to reproduce, a phenomenon observed both in vitro and in vivo. From the underlying stochastic models, transport equations for the cell density is derived.

                These models are qualitatively compared to phase contrast microscopy images of cultivated cells. We make use of Partially Observed Markov Processes to determine the parameters of the individual based model. By considering a parabolic scaling we obtain partial differential equations for the cell density which describes the patterns of growth at the tumour scale. The natural next step after this study is to tackle the same problem using velocity based models, bringing further physical justification to the underlying dynamics and the ability to capture greater detail. The long term ambition of this research is to predict cancer progression in patients, and thus make better informed decisions on how to apply surgery and medical therapy.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Jaeho Shin</h3>
        <p class="text-muted">
          Chalmers University of Technology
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Bio-based production of adipic acid, a nylon ingredient, has been gaining both academic and industrial attention for the last two decades. In this study, we target a theoretical metabolic pathway that relies on conversion of L-lysine to adipic acid. One of the enzymatic steps in this conversion pathway is an α, β-reduction of an unsaturated bond in an enoate moiety and no aerobic enzymes have been identified to make this conversion on 6-amino-trans-2-hexenoic acid. We evaluate Escherichia coli NemA, and Saccharomyces pastorianus Oye1 (Old Yellow Enzyme 1) for their potential capability to carry out the desired α, β-reduction. Here, we build homology models for E. coli NemA and carry out molecular docking studies of trans-2-hexenoic acid and trans-2-hexenal, as model substrates, to the candidate enzyme models including NemA, and Oye1. Ligand-enzyme binding stability is assessed by molecular dynamics simulations. Docking studies accompanied by MD simulations revealed reasonably stable and possible docking conformations. Additionally, linear energy calculations was used to investigate binding stability in solution environment. Here, we propose that NemA from E. coli and Oye1, both belonging to Old yellow enzyme family, have large enough reaction pockets for accommodating enoate moieties but not enough stability to carry out the α, β-reduction. Protein engineering of both NemA and Oye1 would be necessary for these enzymes to perform the targeted reactions efficiently in solution. The results shown in this study provides a useful insight to α, β-reduction reaction in the adipic acid producing pathways.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
</div>
