<div class="row pt-3">
  <div class="col">
    <h1 id="speakers">Invited Speakers</h1>
  </div>
</div>
<div class="row py-1">
  <div class="col-md-8">
    <h2>Matteo Barberis</h2>
    <p class="text-muted">
      Reader/Professor in Systems Biology at the University of Surrey in Guildford, UK
    </p>
    <p style="line-height: 1.1em;">
      <small>
        Matteo has recently become professor at the University of Surrey and has a background in industrial biochemistry, molecular biology and computational modelling. His research focuses on integrating experimentation and computation to gain insight on cellular organization. His main areas of focus are temporal control of gene regulation of cell cycle control and multi-scale frameworks that integrate cellular regulation.
      </small>
    </p>
  </div>
  <div class="col-md-4" style="display: flex; flex-direction: column; justify-content: center;">
    <div class="row">
      <div class="col text-center" style="height: 200px;">
        <img src="https://www.uva.nl/binaries/content/documents/personalpages/b/a/m.barberis/profilepicture/profilepicture/myprofile%3Aimage?1452700579972" alt="Matteo Barberis" class="img-fluid" style="border-radius: 10%; height: 100%;">
      </div>
    </div>
    <div class="row">
      <div class="col text-center">
        <p class="align-middle">
          Photo via
          <a href="https://www.uva.nl/en/profile/b/a/m.barberis/m.barberis.html" target="_blank" class="badge badge-primary mb-3">Personal Website</a>
        </p>
      </div>
    </div>
  </div>
  <div class="col">
    <table class="table">
      <tbody>
        <tr>
          <th scope="row" style="border: none;">Title</th>
          <td style="border: none;">Systems biology to decipher design principles of cellular networks</td>
        </tr>
        <tr>
          <th scape="row" style="border: none;">Abstract</th>
          <td style="border: none;">Continuity of life is sustained by time-dependent responses of biological networks that are associated to oscillatory behavior of their components, at a proper frequency. The eukaryotic cell cycle is such an example, with waves of enzymatic activities being tightly regulated to guarantee its frequency. Therefore, molecular designs that exhibit a proper frequency of enzymatic oscillations are inherently crucial for a timely cell cycle progression. In this talk, I will show how computational frameworks, such as kinetic modelling, Boolean modelling, and multi-scale modelling, may serve the purpose of investigating how a timely cell cycle progression is achieved, possibly resulting in the identification of novel principle of design that ensures a timely cell cycling.</td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<hr>
<div class="row pt-1 pb-3">
  <div class="col-md-8">
    <h2>Francesco Gatto</h2>
    <p class="text-muted">
      Guest Researcher at Chalmers and Co-Founder + Chief Scientific Officer at <a href="https://elypta.com" target="_blank">Elypta</a>
    </p>
    <p style="line-height: 1.1em;">
      <small>
        Named Winner of MIT Technology Review's <em>Innovators Under 35</em> in 2018,
        Francesco co-founded Elypta in 2017, a molecular diagnostics start-up
        focussed to improve the survival outlook of cancer patients by developing
        systems biology-driven biomarkers.
        He invented a diagnostic and prognostic test for renal cell carcinoma based on
        an exceptionally accurate liquid biopsy, which leveraged on a novel systems
        biomarker for this common form of kidney cancer.
        This test recently showed unprecedented potential for other cancer types.
      </small>
    </p>
  </div>
  <div class="col-md-4" style="display: flex; flex-direction: column; justify-content: center;">
    <div class="row">
      <div class="col text-center" style="height: 200px;">
        <img src="https://iu35-prod.s3.amazonaws.com/images/Francesco_Gatto.width-3000.jpg" alt="Francesco Gatto" class="img-fluid" style="border-radius: 10%; height: 100%;">
      </div>
    </div>
    <div class="row">
      <div class="col text-center">
        <p class="align-middle">
          Photo via
          <a href="https://www.innovatorsunder35.com/the-list/francesco-gatto/" target="_blank" class="badge badge-primary mb-3">Innovators Under 35</a>
        </p>
      </div>
    </div>
  </div>
  <div class="col">
    <table class="table">
      <tbody>
        <tr>
          <th scope="row" style="border: none;">Title</th>
          <td style="border: none;">From systems biology research to a diagnostic start-up</td>
        </tr>
        <tr>
          <th scape="row" style="border: none;">Abstract</th>
          <td style="border: none;">An ultimate goal of fundamental research is to improve the life of our people. Systems biology research can help patients to live better and longer by integrating data on their disease into personalized treatment. This is the core of precision medicine and requires innovation in therapeutics and diagnostics. During this talk, I will show an empirical model to make precision medicine a reality, a path set forth by innovation and a path that scientists have a privileged if not an exclusive access to. I will illustrate this model within the context of my own journey with Elypta, a molecular diagnostics start-up I cofounded in 2017 starting from a systems biology research project initiated during my doctoral studies.</td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<hr>
<div class="row pt-1 pb-3">
  <div class="col-md-8">
    <h2>Peter Gennemark</h2>
    <p class="text-muted">
      Principal scientist at <a href="https://www.astrazeneca.com/our-science/gothenburg.html" target="_blank">AstraZeneca R&amp;D</a> and Adjunct Assoc Prof in Systems Biology at Linköping University
    </p>
    <p style="line-height: 1.1em;">
      <small>
        Peter works on mathematical modelling of biological systems and their interactions with drugs (systems biology and pharmacokinetics/pharmacodynamics). He is focusing on systems relevant for metabolic and cardiovascular diseases. In his work, he develops and analyses mathematical models of different granularity, and apply them in the preclinical phase of drug discovery. He collaborates closely on systems biology research topics with Gunnar Cedersund at Linköping University, for instance on modelling of micro-physiological (organ-on-a-chip) systems.
      </small>
    </p>
  </div>
  <div class="col-md-4" style="display: flex; flex-direction: column; justify-content: center;">
    <div class="row">
      <div class="col text-center" style="height: 200px;">
        <img src="https://warwick.ac.uk/fac/sci/eng/research/biomedical/impact/project_members/petergennemarkphoto.jpg" alt="Peter Gennemark" class="img-fluid" style="border-radius: 10%; height: 100%;">
      </div>
    </div>
    <div class="row">
      <div class="col text-center">
        <p class="align-middle">
          Photo via
          <a href="https://warwick.ac.uk/fac/sci/eng/research/biomedical/impact/project_members/" target="_blank" class="badge badge-primary mb-3">IMPACT - Project members</a>
        </p>
      </div>
    </div>
  </div>
  <div class="col">
    <table class="table">
      <tbody>
        <tr>
          <th scope="row" style="border: none;">Title</th>
          <td style="border: none;">Mathematical modelling in preclinical drug discovery</td>
        </tr>
        <tr>
          <th scape="row" style="border: none;">Abstract</th>
          <td style="border: none;">The talk generally describes what type of mathematical models are used in the preclinical phases of drug discovery in the pharmaceutical industry, and how these models are implemented and applied in the discovery process. Focus is on how to translate the pharmacokinetics and the pharmacodynamics of a drug candidate between in vitro systems, animal models and humans. The talk includes several specific examples from the metabolic and cardiovascular areas.</td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<hr>
<div class="row pt-1 pb-3">
  <div class="col-md-8">
    <h2>Adil Mardinoglu</h2>
    <p class="text-muted">
      Professor of Systems Biology at King’s College London, UK and Royal Institute of Technology (KTH), Sweden
    </p>
    <p style="line-height: 1.1em;">
      <small>
        Dr. Adil Mardinoglu works as a Professor of Systems Biology at King’s College London, UK and Royal Institute of Technology (KTH), Sweden. His main area of expertise is in Systems Biology/Medicine, Computational Systems Biology and Bioinformatics (<a href="http://sysmedicine.com" target="_blank">http://sysmedicine.com</a>). His research focuses on the generation of the context specific biological networks for human cells/tissues as well as certain types of cancer including hepatocellular carcinoma by integrating multi-omics data. Such comprehensive networks are employed to identify novel biomarkers and drug targets that can be used for the development of effective treatment strategies for NASH/NAFLD and other obesity-associated disorders. Dr. Mardinoglu is co-founder of two companies focusing on the development of therapies for liver associated diseases.
      </small>
    </p>
  </div>
  <div class="col-md-4" style="display: flex; flex-direction: column; justify-content: center;">
    <div class="row">
      <div class="col text-center" style="height: 200px; overflow: hidden;">
        <img src="https://eas2018.com/wp-content/uploads/2017/04/Mardinoglu500.jpeg" alt="Adil Mardinoglu" class="img-fluid" style="border-radius: 10%; height: 100%;">
      </div>
    </div>
    <div class="row">
      <div class="col text-center">
        <p class="align-middle">
          Photo via
          <a href="https://eas2018.com/" target="_blank" class="badge badge-primary mb-3">86th EAS congress</a>
        </p>
      </div>
    </div>
  </div>
  <div class="col">
    <table class="table">
      <tbody>
        <tr>
          <th scope="row" style="border: none;">Title</th>
          <td style="border: none;">The use of systems biology in treatment of liver diseases</td>
        </tr>
        <tr>
          <th scope="row" style="border: none;">Abstract</th>
          <td style="border: none;">To develop novel strategies for prevention and treatment as well as to gain detailed insights about the underlying molecular mechanisms of liver diseases, it is vital to study the biological functions of liver and its interactions with other tissues and gut microbiota. Biological networks can provide a scaffold for studying biological pathways operating in the liver in connection with disease development in a systematic manner. In my presentation, I will present our recent work where biological networks have been employed to identify the reprogramming in liver physiology in response to NASH/NAFLD. I will further discuss how this mechanistic modelling approach can contribute to the discovery of biomarkers and identification of drug targets which may lead to design of targeted and effective treatment strategies.</td>
        </tr>
        <tr>
          <th scope="row" style="border: none;">Keypoints</th>
          <td style="border: none;">
            <ul>
              <li>Omics technologies are used in detailed characterization of human liver tissue in health and disease states.</li>
              <li>Biological network models are functional tools for exploring and integration of multiomics data.</li>
              <li>Systems biology uses a holistic and integrative approach for comprehensive analysis of the biological functions in healthy and diseased states.</li>
              <li>Systems Biology approaches have been successfully employed in hepatology to identify biomarkers and drug targets.</li>
              <li>These integrative tools can be used for simulation of liver tissue functions and its crosstalk with other tissues for prediction of therapeutic and side effects.</li>
            </ul>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<hr>
<div class="row pt-1 pb-3">
  <div class="col-md-8">
    <h2>Marina Rafajlović</h2>
    <p class="text-muted">
      Assistant Professor at Gothenburg University, Marine Sciences
    </p>
    <p style="line-height: 1.1em;">
      <small>
        Marina has recently become Assistant Professor at the Department of Marine Sciences, University of Gothenburg, and she is a member of the <a href="www.cemeb.science.gu.se" target="_blank">Centre for Marine Evolutionary Biology, University of Gothenburg</a>. Marina has a background in Physics and Theoretical Biology. Her main research interests are within Evolutionary Biology. Specifically, she is interested in the mechanisms of speciation, species' range expansions and the evolutionary advantages of sexual reproduction. Marina's research method involves mathematical modelling (individual-based computer simulations) with model results serving as an input against which empirical genetic data are tested. Marina has recently been awarded a grant by the Hasselblad Foundation for developing her research on the evolution of populations undergoing range shifts, specifically focusing on short- and long-term potential for local genetic adaptation in expanding populations.
      </small>
    </p>
  </div>
  <div class="col-md-4" style="display: flex; flex-direction: column; justify-content: center;">
    <div class="row">
      <div class="col text-center" style="height: 200px;">
        <img src="https://i1.rgstatic.net/ii/profile.image/342395414695938-1458644836948_Q512/Marina_Rafajlovic.jpg" alt="Marina Rafajlovi" class="img-fluid" style="border-radius: 10%; height: 100%;">
      </div>
    </div>
    <div class="row">
      <div class="col text-center">
        <p class="align-middle">
          Photo via
          <a href="https://www.researchgate.net/profile/Marina_Rafajlovic" target="_blank" class="badge badge-primary mb-3">ResearchGate</a>
        </p>
      </div>
    </div>
  </div>
  <div class="col">
    <table class="table">
      <tbody>
        <tr>
          <th scope="row" style="border: none;">Title</th>
          <td style="border: none;">Unfolding evolution using mathematical models</td>
        </tr>
        <tr>
          <th scape="row" style="border: none;">Abstract</th>
          <td style="border: none;">Evolution, the change in the heritable characteristics of biological organisms over successive generations, is the fundamental process responsible for producing and maintaining biodiversity. Over time, an evolving species may face either extinction (e.g. due to the poor potential of its individuals to cope with environmental conditions they encounter), or speciation – the species' split into two (or more) reproductively isolated species (e.g. when individual populations of the species encounter contrasting environmental conditions). Distinguishing the evolutionary mechanims leading to the former or the latter outcome is challenging to date. However, because heriatable characteristics have a genetic basis, patterns of genetic variation within a population are necessarily shaped by the evolutionary history of the population. Therefore, much of the evolutionary mechanism at work in a given population can be learned from empirical patterns of genetic variation in the population. Importantly, to interpret genetic data, it is necessary to produce theoretical expectations for how different evolutionary mechanims and their interplay shape genetic variation during evolution. In this talk, I will highlight some of the recent theoretical advances in the field, specifically focusing on new theoretical insights and open questions in speciation research.</td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<hr>
<div class="row py-3">
  <div class="col">
    <h1>Other speakers</h1>
    <div class="row py-3">
      <div class="col">
        <h3>Gunnar Cedersund</h3>
        <p class="text-muted">
          Linköping University, Department of Biomedical Engineering
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scape="row" style="border: none;">Abstract</th>
              <td style="border: none;">I will in this presentation give an overview of a new technology we have developed for systems medicine, which centers around the concept of a "digital twin". A digital twin is a computer model of of a patient, which has a similar physiology as the patient. This twin can be used for both diagnosis, based on an examination of the patient as he/she is right now, and for prognosis, by doing simulations of how the patient is likely to respond to future treatments, such as medications, exercise, or a change in diet. In all instances, the user can ask the question "why" a certain prediction is obtained, since the model is based on the actual physiological and biochemical processes. Each part of the model has been carefully tested, by an iterative process of data analysis and new experiments, before it has been added to the big picture. The resulting model focuses on the interplay between processes related to metabolism, inflammation, neuronal activity, blood flow, and other processes related to the metabolic syndrome. However, thanks to the physiological structure of the model, it can readily be expanded to include additional processes, if future usages so require. In that sense, this technology goes beyond traditional narrow AI, and lays the foundation for a general AI in healthcare. Apart from healthcare applications, I will also demonstrate how our model is being used in basic research and in drug development.</td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Christoph Börlin</h3>
        <p class="text-muted">
          Chalmers University of Technology
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Title</th>
              <td style="border: none;">
                Predicting gene expression levels from transcription factor binding events through machine learning
              </td>
            </tr>
            <tr>
              <th scape="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Binding of transcription factors (TFs) to gene promoter regions is one of the main drivers of gene regulation, but the exact mechanism and effects are not well understood. As regulation of gene expression is one of the major determinants of cellular phenotype a deeper knowledge of gene regulation is important for understanding cellular behavior and improved metabolic engineering strategies. Many TFs are only active in certain conditions and some TF binding events show no apparent effect on gene expression, making the effects difficult to decipher and to understand. Using the recently developed ChIP-exo method one can identify the binding sites and strengths for a TF with a higher resolution and with less background than previously possible.

                Machine learning approaches have been used in many applications to decipher patterns in large datasets, like speech recognition or predicting cancer treatment outcomes. The large datasets generated by ChIP-exo could also provide a strong basis for a machine learning model linking TF binding events to RNA expression levels. The successful construction of an expression-predictor should then provide valuable insight into the complex interplay of different TFs binding to the same promoter. In our project we collected ChIP-exo data from Saccharomyces cerevisiae for a number of different TFs in four different metabolic conditions. Using tree based machine learning methods we are generating a gene expression predictor that can then be used to predict expression levels for altered promoters in metabolic engineering projects.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Sandro Stucki</h3>
        <p class="text-muted">
          Gothenburg University, Department of Computer Science and Engineering (CSE)
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Title</th>
              <td style="border: none;">
                Rule-Based Modelling of Bio-Chemical Networks
              </td>
            </tr>
            <tr>
              <th scape="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Rule-based modeling (RBM) is a paradigm for describing combinatorially complex stochastic processes with an underlying dynamic network structure through a finite set of rewrite rules. In this talk, I will give an overview of RBM – illustrating how it can be used to model biochemical processes in particular – and present some preliminary work on the automated generation of rate-equations and ODEs tracking higher-order statistics from a large class of rule-based systems.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Anass Bouchnita</h3>
        <p class="text-muted">
          Uppsala University
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scope="row" style="border: none;">Title</th>
              <td style="border: none;">
                Overcoming drug resistance in multiple myeloma: insights from a hybrid discrete-continuous model
              </td>
            </tr>
            <tr>
              <th scope="row" style="border: none;">Authors</th>
              <td style="border: none;">
                Anass Bouchnita, Vitaly Volpert, Mark J. Koury, Andreas Hellander
              </td>
            </tr>
            <tr>
              <th scape="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Drug resistance is one of the main challenges that limit the therapeutic potential of existing cancer treatments. Therefore, designing therapeutic strategies that restore the sensitivity of cancer cells would be greatly beneficial. In this talk, we present a multiscale model devoted to the interaction between multiple myeloma (MM) and potential treatments. In this model, MM cells are represented as individual objects that move, divide, and die by apoptosis. Their fate is determined by intracellular and extracellular regulation, as well as the treatment. We use the model to explore the effects of treatment protocols consisting of a tyrosine-kinase inhibitor (TKI) and a pentose phosphate pathway (PPP) inhibitor. Numerical simulations suggest an interval for the daily dosing of the PPP inhibitor that maximizes the anti-myeloma effects of the TKI. Furthemore, they demonstrate that the combination of high-dose pulsatile TKI treatment with high dose continuous PPP inhibitor therapy can potentially eradicate the tumor. The model predictions can be used to design clinical trials in the future.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Niek Welkenhuysen</h3>
        <p class="text-muted">
          Chalmers University of Technology
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scape="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                Crosstalk in biology is any phenomenon by which an intracellular signal transmitted in one signalling pathway creates an effect in another pathway. Typically, to study these pathway interconnections perturbation of the signalling system is involved. Unfortunately, perturbations often produce noise which causes a major challenge in identification of interconnections. Employing systems biology approach by developing appropriate models can provide minimal intervention in these systems. Typically, signalling pathways are modelled by ordinary differential equations. However, creating dynamical models of signalling pathways of a realistic size is still an obstacle. To overcome this problem, we developed a vector-based reaction-contingency Boolean logic model of the yeast Saccharomyces cerevisiae nutrient sensing pathways cAMP-PKA, Snf1, Snf3-Rgt2 and TOR pathway. To increase the information content of Boolean model from simple 'active' and 'inactive', we assigned a vector to each component describing following features: localization, phosphorylation status, guanylation status and DNA binding status. We found during the gap filling process that most lacking components are phosphatases which exposes a lack of knowledge on phosphatases involved in the sensing process. The gap filling process also identified crosstalk from the PKA and Snf1 pathway to other pathways as a vital aspect to make the model switch between nutrient conditions. We simulated the model with known crosstalk combinations. Subsequent analysis of the simulations that the crosstalk from the Snf1 pathway to the Rgt2/Snf3 pathway contributes to the robustness of this signalling network. Altogether, this work shows that network interconnections lead to the robustness of signalling pathways.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div class="row py-3">
      <div class="col">
        <h3>Annikka Polster</h3>
        <p class="text-muted">
          University of Örebro
        </p>
        <table class="table">
          <tbody>
            <tr>
              <th scape="row" style="border: none;">Abstract</th>
              <td style="border: none;">
                A growing body of evidence confirms the important role gut microbiota have in health and disease, but the detailed mechanisms of interaction between gut microbiota and host are unknown. Given the difficulties in experimentally studying these interactions both in vivo and in vitro, computational approaches useful for determining which microbial species interact with which host factors may greatly enhance our understanding of this interaction and identify which mechanisms are most promising targets for detailed experimental studies. We have developed a stepwise analysis pipeline, in which rank correlations between microbial species and host factors are utilized as a proxy for estimating the grade of interaction between the respective variables, and plotted as network graphs to facilitate the intuitive interpretation of these interactions. The relevance of the respective interaction in the pathophysiology of diseases is estimated by both statistical testing and comparisons between healthy and diseased individuals, as well as by creating a multiplex network graph associating network motifs to key symptoms.

                We have tested the applicability of this pipeline in a pilot study on humans, utilizing data from healthy individuals (HC) as well as patients with Irritable Bowel Syndrome (IBS), a gastrointestinal disorder characterized by various chronic symptoms and incompletely understood pathophysiology.

                In this pilot study, network motifs, indicating close associations of the respective variables, were identified in both groups. A comparison between HC and IBS showed distinct differences in network structure, indicating differences in host-microbiota interaction between these two groups. Network motifs which were present in IBS but not in HC were additionally linked to key symptoms, supporting the relevance of these variables for understanding IBS pathophysiology. This pilot study therefore confirmed our stepwise integrative analysis pipeline as a useful approach for improving our understanding of host-microbiota interaction in health and disease.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
</div>
